<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0069)http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="carcinog"><meta name="ncbi_report" content="printable"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3072670/?report=printable"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/jig.min.css"><script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/jig.min.js"></script><style type="text/css"></style>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of Carcinogenesis"><meta name="citation_title" content="Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development"><meta name="citation_authors" content="Jesús Lascorz, Kari Hemminki, Asta Försti"><meta name="citation_date" content="2011"><meta name="citation_volume" content="10"><meta name="citation_doi" content="10.4103/1477-3163.78268"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3072670/?report=abstract"><meta name="citation_pmid" content="21483658"><meta name="DC.Title" content="Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Medknow Publications"><meta name="DC.Contributor" content="Jesús Lascorz"><meta name="DC.Contributor" content="Kari Hemminki"><meta name="DC.Contributor" content="Asta Försti"><meta name="DC.Date" content="2011"><meta name="DC.Identifier" content="10.4103/1477-3163.78268"><meta name="DC.Language" content="en"><meta property="og:title" content="Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development"><meta property="og:type" content="article"><meta property="og:description" content="A large number of gene expression profiling (GEP) studies on colorectal carcinogenesis have been performed but no reliable gene signature has been identified so far due to the lack of reproducibility in the reported genes. There is growing evidence that ..."><meta property="og:url" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="http://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/common.min.js">//</script><script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:&#39;/pmc/utils/ctxp/&#39;"><script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/jquery.citationexporter.js">//</script><link rel="stylesheet" href="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/citationexporter.css" type="text/css"><script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/MathJax.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-carcinog.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

        <!-- Page meta end -->
    <link rel="shortcut icon" href="http://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="F4FC26677042410100000000006CD1D8"><link type="text/css" rel="stylesheet" href="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/3964959.css"><link type="text/css" rel="stylesheet" href="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/3882866.css" media="print"><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('http://www.ncbi.nlm.nih.gov/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC3072670%2F%3Freport%3Dprintable&ncbi_pdid=article&ncbi_phid=F4FC26677042410100000000006CD1D8'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article hasGoogleVoiceExt" id="ui-jig-5"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            
                            <div class="header">
    
    
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <!-- Journal banner -->
    <div class="pmc-page-banner whole_rhythm"><div><img src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/logo-carcinog.gif" alt="Logo of carcinog" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="poly" coords="0,0,500,0,500,21,407,21,407,75,0,75" alt="Home" title="Home" href="http://www.carcinogenesis.com/" ref="reftype=publisher&amp;article-id=3072670&amp;issue-id=193794&amp;journal-id=138&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="412,24,498,37" alt="Browse articles" title="Browse articles" href="http://www.carcinogenesis.com/browse.asp" ref="reftype=publisher&amp;article-id=3072670&amp;issue-id=193794&amp;journal-id=138&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="410,42,499,53" alt="Instructions" title="Instructions" href="http://www.carcinogenesis.com/contributors.asp" ref="reftype=publisher&amp;article-id=3072670&amp;issue-id=193794&amp;journal-id=138&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="410,59,499,69" alt="Submit articles" title="Submit articles" href="http://www.journalonweb.com/jcar" ref="reftype=publisher&amp;article-id=3072670&amp;issue-id=193794&amp;journal-id=138&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
    
    <div class="">
        
        <div class="hide-overflow article lit-style content pmc-wm slang-all page-box printable"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;" id="ui-jig-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><div><span class="cit">J Carcinog. 2011; 10: 7. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Mar 24. </span>  <span class="doi">doi:&nbsp; <a href="http://dx.doi.org/10.4103%2F1477-3163.78268" target="pmc_ext" ref="reftype=other&amp;article-id=3072670&amp;issue-id=193794&amp;journal-id=138&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.4103/1477-3163.78268</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3072670</span></div></div></div></div></div><h1 class="content-title">Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Lascorz%20J%5Bauth%5D">Jesús Lascorz</a>,<sup>1,</sup><sup>*</sup> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Hemminki%20K%5Bauth%5D">Kari Hemminki</a>,<sup>1,</sup><sup>2</sup> and  <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=F%26%23x000f6%3Brsti%20A%5Bauth%5D">Asta Försti</a><sup>1,</sup><sup>2</sup></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel half_rhythm"><div class="fm-affl" lang="en"><sup>1</sup>Division of Molecular Genetic Epidemiology, German Cancer Research Center (DKFZ), Heidelberg, Germany.</div><div class="fm-affl" lang="en"><sup>2</sup>Center for Primary Health Care Research, Clinical Research Center, Lund University, Malmö, Sweden</div><div><span class="contrib-email" id="__contribid963090">Jesús Lascorz: <a href="https://mail.google.com/mail/?view=cm&amp;fs=1&amp;tf=1&amp;to=dev@null" data-email="ed.zfkd@zrocsal.j" class="oemail" target="_blank">ed.zfkd@zrocsal.j</a></span> </div><div id="cor1"><sup>*</sup>Corresponding author</div></div><div class="fm-article-notes fm-panel half_rhythm"><div class="fm-pubdate half_rhythm">Received 2010 Dec 14; Accepted 2011 Feb 22.</div></div><div class="fm-cpl-info fm-panel half_rhythm"><div class="fm-copyright half_rhythm"><a href="http://www.ncbi.nlm.nih.gov/pmc/about/copyright.html">Copyright</a> © 2011 Lascorz</div><div class="fm-copyright half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution-Noncommercial-Share Alike 3.0 Unported, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div><div class="sec"></div><div id="__abstractid883160" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-6"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true" id="ui-jig-7">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__abstractid883160title">Abstract</h2><!--article-meta--><div><div id="st1" class="sec sec-first"><h3>Background:</h3><p id="__p1" class="p p-first-last">A large number of gene expression profiling (GEP) studies on colorectal carcinogenesis have been performed but no reliable gene signature has been identified so far due to the lack of reproducibility in the reported genes. There is growing evidence that functionally related genes, rather than individual genes, contribute to the etiology of complex traits. We used, as a novel approach, pathway enrichment tools to define functionally related genes that are consistently up- or down-regulated in colorectal carcinogenesis.</p></div><div id="st2" class="sec"><h3> Materials and Methods:</h3><p id="__p2" class="p p-first-last">We started the analysis with 242 unique annotated genes that had been reported by any of three recent meta-analyses covering GEP studies on genes differentially expressed in carcinoma vs normal mucosa. Most of these genes (218, 91.9%) had been reported in at least three GEP studies. These 242 genes were submitted to bioinformatic analysis using a total of nine tools to detect enrichment of Gene Ontology (GO) categories or Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways. As a final consistency criterion the pathway categories had to be enriched by several tools to be taken into consideration.</p></div><div id="st3" class="sec"><h3> Results:</h3><p id="__p3" class="p p-first-last">Our pathway-based enrichment analysis identified the categories of ribosomal protein constituents, extracellular matrix receptor interaction, carbonic anhydrase isozymes, and a general category related to inflammation and cellular response as significantly and consistently overrepresented entities.</p></div><div id="st4" class="sec sec-last"><h3> Conclusions:</h3><p id="__p4" class="p p-first-last">We triaged the genes covered by the published GEP literature on colorectal carcinogenesis and subjected them to multiple enrichment tools in order to identify the consistently enriched gene categories. These turned out to have known functional relationships to cancer development and thus deserve further investigation.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Carcinogenesis, colorectal cancer, enrichment analysis, gene expression profiling</span></div></div><div id="sec1-1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-8"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-9">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-1title">BACKGROUND</h2><p id="__p5" class="p p-first">Colorectal cancer (CRC) is the third most common cancer, comprising 9.7% of all cancer cases, and is the fourth leading cause of cancer death worldwide, accounting for 8% of all cancer deaths.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref1" rid="ref1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318491358">1</a>] Many gene expression profiling (GEP) studies on colorectal carcinogenesis have been performed in the last decade using microarray technology. However, comparative analysis of the differentially expressed genes reported by independent studies shows a relatively limited degree of overlap, and no reliable biomarker profile discriminating cancerous from normal tissue has been identified. The majority of the published GEP studies on colorectal carcinogenesis has already been subjected to meta-analyses that have aimed at establishing consistent signature profiles for tumor development.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref2" rid="ref2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534024">2</a>–<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref4" rid="ref4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534032">4</a>] These meta-analyses have collected published lists of differentially expressed genes from the original GEP studies comparing CRC to normal tissue and then selected the genes reported in multiple studies. The genes reported only sporadically are thought to have resulted from inherent noise or biases in the different platforms and analysis methods employed.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref5" rid="ref5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534042">5</a>] The consistently reported genes are considered to be biologically relevant to CRC.</p><p id="__p6">There is an increasing interest in searching for networks of genes, instead of single genes, contributing to the etiology of complex diseases, since changes in biological characteristics require coordinate variation in expression of gene sets.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref6" rid="ref6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_271263901">6</a>] Enrichment analysis tools, which estimate overrepresentation of particular gene categories or pathways in a gene list, are a useful approach in this direction.</p><p id="__p7" class="p p-last">Our goal was to define functional categories [Gene Ontology (GO) terms or Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways] that are consistently overrepresented among differentially expressed genes inferred from the published GEP studies on colorectal carcinogenesis. We collected the list of genes from three published meta-analyses and used them as an input list for an overrepresentation analysis with several independent enrichment tools, which are based on diverse statistical and bioinformatic algorithms.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref7" rid="ref7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534043">7</a>] The strategy of applying multiple tools is recommended for the most satisfactory results.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref8" rid="ref8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534029">8</a>] The stringent selection criteria for the genes to be analyzed and the requirement for concordance between enrichment analysis results helped us to identify consistently enriched gene categories of likely relevance in colorectal carcinogenesis.</p></div><div id="sec1-2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-10"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-11">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-2title">MATERIALS AND METHODS</h2><div id="sec2-1" class="sec sec-first"><h3>Gene expression profiling studies</h3><p id="__p8" class="p p-first">We collected data from three meta-analyses, covering 34 GEP studies on the colorectal carcinogenesis process, published between the years 2001 and 2007.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref2" rid="ref2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534030">2</a>–<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref4" rid="ref4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534040">4</a>] Two of the meta-analyses reported a list of genes which had a consistent direction in gene expression change between carcinoma and normal mucosa in at least three single GEP studies,[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref2" rid="ref2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534021">2</a>,<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref3" rid="ref3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534023">3</a>] while the threshold was two GEP studies in the oldest meta-analysis[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref4" rid="ref4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534027">4</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>].</p></div><div id="sec2-2" class="sec"><h3>Gene list collection</h3><p id="__p10" class="p p-first">For the meta-analysis by Sagynaliev <em>et al</em>.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref4" rid="ref4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534019">4</a>] we used Entrez Gene from NCBI (www.ncbi.nlm.nih.gov/gene/), and the Gene ID conversion tool from the DAVID bioinformatics resources[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref9" rid="ref9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534035">9</a>] to convert the reported gene identifiers into the official HUGO gene symbol, which was used as the identifier for the reported genes. Next, the three gene lists from the three meta-analyses were combined, resulting in a list of 242 unique annotated genes [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>].</p></div><div id="sec2-3" class="sec"><h3>Enrichment analysis</h3><p id="__p12" class="p p-first">We performed enrichment analyses using the databases GO (Biological Process and Molecular Function),[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref10" rid="ref10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534020">10</a>] and KEGG pathways.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref11" rid="ref11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534033">11</a>] For all enrichment tools, the input gene set consisted of the same 242-gene list. The nine selected enrichment software tools differed in the statistical model applied for the enrichment analysis and in the method of correction for multiple testing [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T3/" target="true" class="fig-table-link table figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>]. The tools were used with the default options: significance threshold of 0.05 for adjusted <em>P</em> value, at least two genes from the input list in the enriched category, and the whole genome as the reference background. For GATHER, the recommended ln(Bayes factor) &gt;6 was used as the significance threshold.</p></div><div id="sec2-4" class="sec sec-last"><h3>Consistently enriched categories</h3><p id="__p14" class="p p-first">We considered only the GO or KEGG categories reported to be significantly enriched by several enrichment tools as consistently overrepresented in the 242-gene list. This strategy, based on testing multiple tools, is recommended in order to obtain the most satisfactory results.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref8" rid="ref8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534039">8</a>] We selected as a threshold the number of tools reporting at least four common enriched categories, so that only top-ranked categories were finally considered. This threshold was five enrichment tools for GO Biological Process, six enrichment tools for GO Molecular Function, and three enrichment tools for KEGG pathways [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T4/" target="true" class="fig-table-link table figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>].</p></div></div><div id="sec1-3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-12"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-13">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-3title">RESULTS</h2><div id="sec2-5" class="sec sec-first"><h3>Data collection and gene selection</h3><p id="__p16" class="p p-first-last">A total of 242 unique mapped genes [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T2/" target="true" class="fig-table-link table figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>] were reported in at least one of the three meta-analyses (65 of them in two and 26 in all three meta-analyses), 145 (59.9%) of the genes were up-regulated and 97 (40.1%) down-regulated in cancer vs normal tissue. Twenty-four of the 242 genes (9.9%) had been reported by two single GEP studies and 218 genes (90.1%) by at least three single GEP studies.</p></div><div id="sec2-6" class="sec"><h3>Enrichment analyses</h3><p id="__p17" class="p p-first">Nine enrichment tools were used to obtain significantly overrepresented categories (GO Biological Process, GO Molecular Function, and KEGG pathways) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T5/" target="true" class="fig-table-link table figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>].</p></div><div id="sec2-7" class="sec sec-last"><h3>Identification of consistently enriched categories</h3><p id="__p21" class="p p-first">The number of reported enriched categories showed considerable variability with the different tools used [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T4/" target="true" class="fig-table-link table figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>] even though the same significance threshold (<em>P</em>&lt;.05 after correction for multiple testing) and analysis conditions (whole genome as the reference background and at least two genes from the input list in the enriched category) were applied. Differences were also observed in the number of genes in a particular category and the enrichment <em>P</em> values reported by each tool [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T5/" target="true" class="fig-table-link table figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>]. To avoid false positives among the varying results, only the categories reported to be enriched by several tools (five enrichment tools for GO Biological Process, six for GO Molecular Function, and three for KEGG pathways) were considered to be consistently enriched. Using this selection criteria, ten general GO Biological Process categories (<em>cell proliferation, inflammatory response, multicellular organismal metabolic process, regulation of cell proliferation, response to chemical stimulus, response to external stimulus, response to nutrient, response to stress, response to wounding, and translational elongation</em>); five GO Molecular Function categories (<em>carbonate dehydratase activity, cytokine activity, extracellular matrix binding, receptor binding, and structural constituent of ribosome</em>); and four KEGG pathways (<em>extracellular matrix receptor interaction, focal adhesion, nitrogen metabolism, and ribosome</em>) were consistently overrepresented in the 242 gene list [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T8/" target="true" class="fig-table-link table figpopup" rid-figpopup="T8" rid-ob="ob-T8" co-legend-rid=""><span>Table 6</span></a>]. The ratio of enrichment was higher for the more specific and well-defined KEGG pathways than for the broad GO categories [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>]. A very high overlap of the individual genes among these categories was also observed [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/table/T9/" target="true" class="fig-table-link table figpopup" rid-figpopup="T9" rid-ob="ob-T9" co-legend-rid=""><span>Table 7</span></a>]. Based on this overlap, four biologically meaningful category groups were finally obtained:</p><div id="__p25"><p>
</p><ol class="enumerated" style="list-style-type:lower-alpha"><!--
list-behavior=enumerated
prefix-word=
mark-type=lower-alpha
--><li><div>Seventeen common genes included in the GO Biological Process <em>translational elongation</em>, the GO Molecular Function <em>structural constituent of ribosome</em>, and the KEGG pathway <em>ribosome</em>.</div></li><li><div>Genes in the two KEGG pathways <em>extracellular matrix receptor interaction</em> and <em>focal adhesion</em> that were also included in the broad categories of GO Molecular Function <em>receptor binding</em> and GO Biological Process <em>response to external stimulus</em>.</div></li><li><div>The five genes included in both the GO Molecular Function category <em>carbonate dehydratase activity</em> and the KEGG pathway <em>nitrogen metabolism</em>.</div></li><li><div>A large group of seven general GO Biological Process categories (<em>inflammatory response, response to chemical stimulus, response to external stimulus, response to nutrient, response to stress,</em> and <em>response to wounding</em>), together with two general GO Molecular Function categories (<em>cytokine activity</em> and <em>receptor binding</em>).</div></li></ol><p>
</p></div></div></div><div id="sec1-4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-14"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-15">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-4title">DISCUSSION</h2><p id="__p30" class="p p-first">The large number of microarray studies on colorectal carcinogenesis has shown a low degree of overlap in the identified genes. We extracted the 242 unique genes reported in three meta-analyses of GEP studies on colorectal carcinogenesis.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref2" rid="ref2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534017">2</a>–<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref4" rid="ref4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534026">4</a>] Only the meta-analysis by Cardoso <em>et al</em>.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref2" rid="ref2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534037">2</a>] includes a descriptive exploration of the main GO categories present among the differentially expressed genes. In an attempt to overcome the known lack of reproducibility at individual gene level among the GEP studies, we used up to nine bioinformatic enrichment tools to statistically determine which GO categories or KEGG pathways were significantly overrepresented in the 242-gene list. A total of 34 independent GEP studies were included in the three meta-analyses. Most of them used whole-genome expression arrays, which include probes for expression analysis of thousands of genes. Thus, we used all genes in the genome as background for the enrichment analysis. Although this might be an overestimation, the heterogeneity in the number of genes interrogated in every single one of the 34 GEP experiments does not allow application of a more appropriate restricted background. We believe that our rigorous strategy for the selection of enriched categories overcomes the forced probable overestimation of the reference background. After application of rigorous selection criteria, a total of 19 categories (15 GO terms and 4 KEGG pathways) were considered as consistently overrepresented. When considering the individual genes from each of these 19 categories, a very high degree of overlap among the categories was observed, reducing the number of categories with biological significance to four clearly different groups.</p><p id="__p31">First, the same 17 ribosomal proteins (RPs) were present in the GO Biological Process <em>translational elongation</em>, the GO Molecular Function <em>structural constituent of ribosome</em>, and the KEGG pathway <em>ribosome</em> (RPL3, RPL6, RPL7, RPL8, RPL18A, RPL23, RPL29, RPL30, RPL31, RPLP2, RPSA, RPS2, RPS5, RPS7, RPS18, RPS19, and RPS23) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>]. All of them showed increased expression in tumor <em>vs</em> normal tissue. It is known that different expression patterns of RPs exist in CRC. Also, ribosomal biogenesis has clearly been linked to cancer[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref12" rid="ref12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534031">12</a>] and several studies have pointed out two possible functions of RPs in colorectal carcinogenesis: perturbation of their function in protein biosynthesis and direct influence in tumorigenesis through extraribosomal functions (summarized in Lai <em>et al</em>.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref13" rid="ref13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534045">13</a>] ). Second, the KEGG terms <em>extracellular matrix receptor interaction</em> and <em>focal adhesion</em> shared nine genes (COL1A1, COL1A2, COL3A1, COL4A1, COL11A1, FN1, ITGA2, SPP1, and THBS2) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>]. Specific interactions of the extracellular matrix molecules control cellular activities such as adhesion, differentiation, apoptosis, and proliferation.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref14" rid="ref14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_495831114">14</a>] Third, the GO category <em>carbonate dehydratase activity</em> and the KEGG pathway <em>nitrogen metabolism</em> included the same five carbonic anhydrase (CA) isozymes (CA1, CA2, CA4, CA7, and CA12) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>]. All five mRNAs are down-regulated in CRC compared to normal tissue, as also shown in another study for CA2 and CA12.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref15" rid="ref15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534041">15</a>] Recent data have confirmed the functional contribution of CAs, especially CA9 and CA12, to hypoxic tumor growth and progression.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref16" rid="ref16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534018">16</a>] Inhibition of CA9, which is overexpressed in many tumor types in response to the hypoxia inducible factor (HIF) pathway, is being tested as anticancer therapeutic strategy.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref17" rid="ref17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534036">17</a>] Finally, a very general group of GO categories related to inflammation and cellular response included a large number of genes (between 14 and 59). Interestingly, this category included two genes that have been identified through genome-wide association studies as low-risk inherited genetic variants contributing to CRC risk.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref18" rid="ref18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534025">18</a>] These genes, the proto-oncogene MYC (8q24) and the bone morphogenetic protein gene BMP4 (14q22.2), were up-regulated in carcinoma tissue. Thus, judging by the functional class of the genes from the identified enriched categories, they look promising candidates for studies aimed at investigating their possible influence in CRC development.</p><p id="__p35" class="p p-last">In general, we observed a considerable variation in the number of enriched categories reported by each tool although there was uniformity in the analysis conditions used. However, despite this apparent variation, most of the enriched categories reported by the more stringent tools (those reporting a small number of enriched categories) were ranked among the top-categories by the more generous tools (those reporting a larger number of enriched categories). We considered this result of special interest because of previously reported lack of reproducibility between different enrichment tools.[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref7" rid="ref7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534028">7</a>,<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref8" rid="ref8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534034">8</a>,<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#ref19" rid="ref19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_216534022">19</a>] This variability has been attributed to the statistical models applied by the enrichment analysis, to the method of correction for multiple testing, and to differences in the versions of the GO and KEGG data sources used. Thus, our strategy of using several bioinformatic tools to extract biologically related genes consistently involved in colorectal carcinogenesis proved to be successful.</p></div><div id="sec1-5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-16"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-17">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-5title">CONCLUSIONS</h2><p id="__p36" class="p p-first-last">We used the list of 242 unique mapped genes from three meta-analyses of GEP studies on colorectal carcinogenesis for a systematic enrichment analysis of GO categories and KEGG pathways, applying up to nine different enrichment tools. After applying stringent selection criteria to avoid false positive results, the ribosomal proteins group, the extracellular matrix receptor interaction category, the carbonic anhydrase isozymes, and a general category related to inflammation emerged as significantly and consistently overrepresented categories. These categories have known functional relationships to CRC development and their value as diagnostic markers and therapeutic targets deserve further investigation.</p></div><div id="sec1-6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-18"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-19">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-6title">Competing interests</h2><p id="__p37" class="p p-first-last">The authors declare that they have no competing interests.</p></div><div id="sec1-7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-20"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-21">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-7title">Authors’ contributions</h2><p id="__p38" class="p p-first-last">JL and AF conceived and designed the study. JL conducted the analyses and wrote the initial manuscript. KH provided oversight and conceptual guidance to the project. KH and AF contributed to the final manuscript. All authors read and approved the final manuscript.</p></div><div id="sec1-8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-22"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-23">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-8title">AUTHOR'S PROFILE</h2><p id="__p39" class="p p-first"><strong>Prof. Kari Hemminki,</strong> 1973 PhD in Medicine, University of Helsinki, Finland 1973
MD in Medicine, University of Helsinki, Finland 1975 Docent in Biochemistry,
University of Helsinki, Finland 1976 - 1978 Postdoc in Molecular Biology, John
Hopkins, Baltimore, USA.</p><p id="__p40"><strong>Jesus Lascorz,</strong> 2001 BsC in Biochemistry, University of Zaragoza, Spain 2002
- 2003 Molecular Genetics Laboratory, Central Institute of Mental Health,
University of Heidelberg, Mannheim, Germany 2008 PhD in Human Biology,
Institute of Human Genetics, University Erlangen-Nürnberg, Germany</p><p id="__p41"><strong>Asta Foersti,</strong> 1984 M.Sc. in Biochemistry, University of Kuopio, Finland 1992 PhD
in Biochemistry and Biotechnology, University of Kuopio, Finland</p><p id="__p42" class="p p-last"><strong>Dr. Jason A. Zell,</strong> is Assistant Professor of Medicine (Hematology/Oncology) and Epidemiology at the School of Medicine, University of California Irvine</p></div><div id="__fn-groupid927501" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-24"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-25">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__fn-groupid927501title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="__fnid927502"></p><p id="__p43" class="p p-first-last"><strong>Funding</strong> German National Genome Research Network (NGFN-Plus); the Deutsche Krebshilfe (German Cancer AID); and European Union (EU) (HEALTH-F4-2007-200767).</p><p></p></div></div><div id="__ref-listid927514" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-26"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-27">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__ref-listid927514title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ref1">1. <span class="element-citation">Ferlay J, Shin HR, Bray F, Forman D, Mathers C, Parkin DM. Estimates of worldwide burden of cancer in 2008: GLOBOCAN 2008. <span><span class="ref-journal">Int J Cancer. </span>2010</span> in press.  <span class="nowrap ref pubmed"></span></span></div><div class="ref-cit-blk half_rhythm" id="ref2">2. <span class="element-citation">Cardoso J, Boer J, Morreau H, Fodde R. Expression and genomic profiling of colorectal cancer. <span><span class="ref-journal">Biochim Biophys Acta. </span>2007;<span class="ref-vol">1775</span>:103–37.</span>  <span class="nowrap ref pubmed">[PubMed: 17010523]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref3">3. <span class="element-citation">Chan SK, Griffith OL, Tai IT, Jones SJ. Meta-analysis of colorectal cancer gene expression profiling studies identifies consistently reported candidate biomarkers. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2008;<span class="ref-vol">17</span>:543–52.</span>  <span class="nowrap ref pubmed">[PubMed: 18349271]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref4">4. <span class="element-citation">Sagynaliev E, Steinert R, Nestler G, Lippert H, Knoch M, Reymond MA. Web-based data warehouse on gene expression in human colorectal cancer. <span><span class="ref-journal">Proteomics. </span>2005;<span class="ref-vol">5</span>:3066–78.</span>  <span class="nowrap ref pubmed">[PubMed: 16041676]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref5">5. <span class="element-citation">Siddiqui AS, Delaney AD, Schnerch A, Griffith OL, Jones SJ, Marra MA. Sequence biases in large scale gene expression profiling data. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:e83.</span> <span class="nowrap ref pmc">[PMCID: PMC1524917]</span>  <span class="nowrap ref pubmed">[PubMed: 16840527]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref6">6. <span class="element-citation">Hardy J, Singleton A. Genomewide association studies and human disease. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">360</span>:1759–68.</span> <span class="nowrap ref pmc">[PMCID: PMC3422859]</span>  <span class="nowrap ref pubmed">[PubMed: 19369657]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref7">7. <span class="element-citation">Huang da W, Sherman BT, Lempicki RA. Bioinformatics enrichment tools: Paths toward the comprehensive functional analysis of large gene lists. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>:1–13.</span> <span class="nowrap ref pmc">[PMCID: PMC2615629]</span>  <span class="nowrap ref pubmed">[PubMed: 19033363]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref8">8. <span class="element-citation">Rhee SY, Wood V, Dolinski K, Draghici S. Use and misuse of the gene ontology annotations. <span><span class="ref-journal">Nat Rev Genet. </span>2008;<span class="ref-vol">9</span>:509–15.</span>  <span class="nowrap ref pubmed">[PubMed: 18475267]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref9">9. <span class="element-citation">Huang da W, Sherman BT, Lempicki RA. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. <span><span class="ref-journal">Nat Protoc. </span>2009;<span class="ref-vol">4</span>:44–57.</span>  <span class="nowrap ref pubmed">[PubMed: 19131956]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref10">10. <span class="element-citation">Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et al.  Gene ontology: Tool for the unification of biology: The Gene Ontology Consortium. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">25</span>:25–9.</span> <span class="nowrap ref pmc">[PMCID: PMC3037419]</span>  <span class="nowrap ref pubmed">[PubMed: 10802651]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref11">11. <span class="element-citation">Kanehisa M, Goto S, Kawashima S, Okuno Y, Hattori M. The KEGG resource for deciphering the genome. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32(Database issue)</span>:D277–80.</span> <span class="nowrap ref pmc">[PMCID: PMC308797]</span>  <span class="nowrap ref pubmed">[PubMed: 14681412]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref12">12. <span class="element-citation">Lempiainen H, Shore D. Growth control and ribosome biogenesis. <span><span class="ref-journal">Curr Opin Cell Biol. </span>2009;<span class="ref-vol">21</span>:855–63.</span>  <span class="nowrap ref pubmed">[PubMed: 19796927]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref13">13. <span class="element-citation">Lai MD, Xu J. Ribosomal proteins and colorectal cancer. <span><span class="ref-journal">Curr Genomics. </span>2007;<span class="ref-vol">8</span>:43–9.</span> <span class="nowrap ref pmc">[PMCID: PMC2474683]</span>  <span class="nowrap ref pubmed">[PubMed: 18645623]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref14">14. <span class="element-citation">Desgrosellier JS, Cheresh DA. Integrins in cancer: Biological implications and therapeutic opportunities. <span><span class="ref-journal">Nat Rev Cancer. </span>2010;<span class="ref-vol">10</span>:9–22.</span> <span class="nowrap ref pmc">[PMCID: PMC4383089]</span>  <span class="nowrap ref pubmed">[PubMed: 20029421]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref15">15. <span class="element-citation">Niemela AM, Hynninen P, Mecklin JP, Kuopio T, Kokko A, Aaltonen L, et al.  Carbonic anhydrase IX is highly expressed in hereditary nonpolyposis colorectal cancer. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2007;<span class="ref-vol">16</span>:1760–6.</span>  <span class="nowrap ref pubmed">[PubMed: 17855694]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref16">16. <span class="element-citation">Guler OO, De Simone G, Supuran CT. Drug design studies of the novel antitumor targets carbonic anhydrase IX and XII. <span><span class="ref-journal">Curr Med Chem. </span>2010;<span class="ref-vol">17</span>:1516–26.</span>  <span class="nowrap ref pubmed">[PubMed: 20166929]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref17">17. <span class="element-citation">Poulsen SA. Carbonic anhydrase inhibition as a cancer therapy: A review of patent literature, 2007 - 2009. <span><span class="ref-journal">Expert Opin Ther Pat. </span>2010;<span class="ref-vol">20</span>:795–806.</span>  <span class="nowrap ref pubmed">[PubMed: 20476848]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref18">18. <span class="element-citation">Tenesa A, Dunlop MG. New insights into the aetiology of colorectal cancer from genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2009;<span class="ref-vol">10</span>:353–8.</span>  <span class="nowrap ref pubmed">[PubMed: 19434079]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref19">19. <span class="element-citation">Khatri P, Draghici S. Ontological analysis of gene expression data: Current tools, limitations, and open problems. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:3587–95.</span> <span class="nowrap ref pmc">[PMCID: PMC2435250]</span>  <span class="nowrap ref pubmed">[PubMed: 15994189]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref20">20. <span class="element-citation">Kamburov A, Wierling C, Lehrach H, Herwig R. ConsensusPathDB–a database for integrating human functional interaction networks. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>(Database issue):D623–8.</span> <span class="nowrap ref pmc">[PMCID: PMC2686562]</span>  <span class="nowrap ref pubmed">[PubMed: 18940869]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref21">21. <span class="element-citation">Al-Shahrour F, Diaz-Uriarte R, Dopazo J. FatiGO: A web tool for finding significant associations of Gene Ontology terms with groups of genes. <span><span class="ref-journal">Bioinformatics. </span>2004;<span class="ref-vol">20</span>:578–80.</span>  <span class="nowrap ref pubmed">[PubMed: 14990455]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref22">22. <span class="element-citation">Chang JT, Nevins JR. GATHER: A systems approach to interpreting genomic signatures. <span><span class="ref-journal">Bioinformatics. </span>2006;<span class="ref-vol">22</span>:2926–33.</span>  <span class="nowrap ref pubmed">[PubMed: 17000751]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref23">23. <span class="element-citation">Carmona-Saez P, Chagoyen M, Tirado F, Carazo JM, Pascual-Montano A. GENECODIS: A web-based tool for finding significant concurrent annotations in gene lists. <span><span class="ref-journal">Genome Biol. </span>2007;<span class="ref-vol">8</span>:R3.</span> <span class="nowrap ref pmc">[PMCID: PMC1839127]</span>  <span class="nowrap ref pubmed">[PubMed: 17204154]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref24">24. <span class="element-citation">Zhang B, Schmoyer D, Kirov S, Snoddy J. GOTree Machine (GOTM): A web-based platform for interpreting sets of interesting genes using Gene Ontology hierarchies. <span><span class="ref-journal">BMC Bioinformatics. </span>2004;<span class="ref-vol">5</span>:16.</span> <span class="nowrap ref pmc">[PMCID: PMC373441]</span>  <span class="nowrap ref pubmed">[PubMed: 14975175]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref25">25. <span class="element-citation">Reimand J, Kull M, Peterson H, Hansen J, Vilo J. g:Profiler–a web-based toolset for functional profiling of gene lists from large-scale experiments. <span><span class="ref-journal">Nucleic Acids Res. </span>2007;<span class="ref-vol">35</span>:W193–200.</span> <span class="nowrap ref pmc">[PMCID: PMC1933153]</span>  <span class="nowrap ref pubmed">[PubMed: 17478515]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref26">26. <span class="element-citation">Chen J, Bardes EE, Aronow BJ, Jegga AG. ToppGene Suite for gene list enrichment analysis and candidate gene prioritization. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>:W305–11.</span> <span class="nowrap ref pmc">[PMCID: PMC2703978]</span>  <span class="nowrap ref pubmed">[PubMed: 19465376]</span></span></div><div class="ref-cit-blk half_rhythm" id="ref27">27. <span class="element-citation">Zhang B, Kirov S, Snoddy J. WebGestalt: An integrated system for exploring gene sets in various biological contexts. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:W741–8.</span> <span class="nowrap ref pmc">[PMCID: PMC1160236]</span>  <span class="nowrap ref pubmed">[PubMed: 15980575]</span></span></div></div></div><div id="__articleid270543" class="tsec sec print-view"><div class="goto jig-ncbiinpagenav-goto-container" id="ui-jig-28"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true" id="ui-jig-29">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__articleid270543title">Figures and Tables</h2><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g001.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g001.jpg"></div><!--caption a7--><div class="caption"><p id="__p9">Three meta-analyses of gene expression profiling studies on CRC carcinogenesis process</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g002.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g002.jpg"></div><!--caption a7--><div class="caption"><p id="__p11">The list of 242 unique, annotated genes reported in the three meta-analyses of GEP studies on CRC carcinogenesis used for enrichment analyses</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g003.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g003.jpg"></div><!--caption a7--><div class="caption"><p id="__p13">Enrichment tools used and their characteristics</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g004.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g004.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g004.jpg"></div><!--caption a7--><div class="caption"><p id="__p15">Number of overrepresented GO and KEGG categories in the 242-gene list for each of the enrichment tools used</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5A</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g005.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g005.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g005.jpg"></div><!--caption a7--><div class="caption"><p id="__p18">Results of all enrichment tools used with the 242 gene list: Gene ontology biological process categories</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T6"><h3>Table 5B</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g006.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g006.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g006.jpg"></div><!--caption a7--><div class="caption"><p id="__p19">Results of all enrichment tools used with the 242 gene list: Gene ontology molecular function categories</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T7"><h3>Table 5C</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g007.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g007.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g007.jpg"></div><!--caption a7--><div class="caption"><p id="__p20">Results of all enrichment tools used with the 242 gene list: KEGG pathway categories</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T8"><h3>Table 6</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g008.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g008.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g008.jpg"></div><!--caption a7--><div class="caption"><p id="__p22">Consistently enriched GO and KEGG categories</p></div></div></div><div class="print-obj whole_rhythm"><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F1"><h3>Figure 1</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g009.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g009.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g009.jpg"></div><!--caption a7--><div class="caption"><p id="__p23">Bar chart of enrichment ratios for GO and KEGG categories in the 242-gene list. Ratio of enrichment = the number of observed genes divided by the number of expected genes from each GO or KEGG category in the 242-gene list (according to WebGestalt or, alternatively, DAVID or GOTM tools). GO BP: Gene Ontology Biological Process; GO MF: Gene Ontology Molecular Function; KEGG: Kyoto Encyclopedia of Genes and Genomes.</p></div></div></div><div class="print-obj whole_rhythm"><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T9"><h3>Table 7</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g010.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g010.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g010.jpg"></div><!--caption a7--><div class="caption"><p id="__p24">Overlap of the genes from the consistently enriched GO and KEGG categories</p></div></div></div><div class="print-obj whole_rhythm"><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F2"><h3>Figure 2</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g011.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g011.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g011.jpg"></div><!--caption a7--><div class="caption"><p id="__p32">Representation of the KEGG ribosome category (map03010), with the 17 genes from the 242 gene list indicated in red</p></div></div></div><div class="print-obj whole_rhythm"><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F3"><h3>Figure 3</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g012.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g012.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g012.jpg"></div><!--caption a7--><div class="caption"><p id="__p33">Representation of the KEGG extracellular matrix receptor interaction category (map04512), with location of the ten genes from the 242 gene list indicated in red.</p></div></div></div><div class="print-obj whole_rhythm"><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F4"><h3>Figure 4</h3><div class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g013.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JC-10-7-g013.jpg" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/JC-10-7-g013.jpg"></div><!--caption a7--><div class="caption"><p id="__p34">Representation of the KEGG nitrogen metabolism category (map00910), with location of the reaction catalyzed by the five carbonic anhydrase isozymes from the 242 gene list indicated in red</p></div></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#__abstractid883160title">Abstract</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-1title">BACKGROUND</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-2title">MATERIALS AND METHODS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-3title">RESULTS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-4title">DISCUSSION</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-5title">CONCLUSIONS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-6title">Competing interests</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-7title">Authors’ contributions</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#sec1-8title">AUTHOR'S PROFILE</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#__fn-groupid927501title">Footnotes</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#__ref-listid927514title">REFERENCES</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072670/?report=printable#__articleid270543title">Figures and Tables</a></li></ul></div></div><!--post-content--><hr class="whole_rhythm no_bottom_margin"><div class="courtesy-note no_margin small">Articles from <span class="acknowledgment-journal-title">Journal of Carcinogenesis</span> are provided here courtesy of <strong>Medknow Publications</strong></div></div>
        
        
    </div>
</div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    
    
    
</div>

                            <script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/InstrumentPageStarterJS.js"> </script>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=carcinog&amp;amp;ncbi_report=printable&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC3072670/?report=printable&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- F4FB0B9C6B8F76D1_0090SID /projects/PMC/PMCViewer@4.35 ptpmc202 v4.1.r480124 Mon, Sep 28 2015 02:46:19 -->

<script type="text/javascript" src="./Lascorz, Hemminki, Forsti.2011.Systematic enrichment analysis of gene expression profiling studies identifies consensus pathways implicated in colorectal cancer development_files/3821238.js" snapshot="pmc"></script>
<ul class="print-log"><li></li></ul><div id="directions_extension" style="display: none;"></div></body></html>