<!DOCTYPE HTML>
<!-- saved from url=(0077)http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x -->
<!DOCTYPE html PUBLIC "" ""><!--[if lte IE 8 ]><html lang="en" class="no-js oldIE"><![endif]--><!--[if (gt IE 8)|!(IE)]><!--><HTML 
class="no-js" lang="en"><!--<![endif]--><HEAD><META content="IE=11.0000" 
http-equiv="X-UA-Compatible">
     <TITLE>Predicting drug target interactions using meta-path-based semantic 
network analysis | BMC Bioinformatics | Full Text</TITLE>         
<META name="journal_id" content="12859">     
<META name="dc.identifier" content="10.1186/s12859-016-1005-x">     
<META name="dc.title" content="Predicting drug target interactions using meta-path-based semantic network analysis"> 
    
<META name="dc.creator" content="Gang Fu">     
<META name="dc.creator" content="Ying Ding">     
<META name="dc.creator" content="Abhik Seal">     
<META name="dc.creator" content="Bin Chen">     
<META name="dc.creator" content="Yizhou Sun">     
<META name="dc.creator" content="Evan Bolton">     
<META name="dc.description" content="In the context of drug discovery, drug target interactions (DTIs) can be predicted based on observed topological features of a semantic network across the chemical and biological space. In a semantic network, the types of the nodes and links are different. In order to take into account the heterogeneity of the semantic network, meta-path-based topological patterns were investigated for link prediction.Supervised machine learning models were constructed based on meta-path topological features of an enriched semantic network, which was derived from Chem2Bio2RDF, and was expanded by adding compound and protein similarity neighboring links obtained from the PubChem databases. The additional semantic links significantly improved the predictive performance of the supervised learning models. The binary classification model built upon the enriched feature space using the Random Forest algorithm significantly outperformed an existing semantic link prediction algorithm, Semantic Link Association Prediction (SLAP), to predict unknown links between compounds and protein targets in an evolving network. In addition to link prediction, Random Forest also has an intrinsic feature ranking algorithm, which can be used to select the important topological features that contribute to link prediction.The proposed framework has been demonstrated as a powerful alternative to SLAP in order to predict DTIs using the semantic network that integrates chemical, pharmacological, genomic, biological, functional, and biomedical information into a unified framework. It offers the flexibility to enrich the feature space by using different normalization processes on the topological features, and it can perform model construction and feature selection at the same time."> 
    
<META name="dc.source" content="BMC Bioinformatics 2016 17:1">     
<META name="dc.format" content="text/html">     
<META name="dc.publisher" content="BioMed Central">     
<META name="dc.date" content="2016-04-12">     
<META name="dc.type" content="OriginalPaper">     
<META name="dc.language" content="En">     
<META name="dc.copyright" content="2016 Fu et al.">     
<META name="dc.rightsAgent" content="reprints@biomedcentral.com">     
<META name="dc.subject" content="Bioinformatics">     
<META name="dc.subject" content="Microarrays">     
<META name="dc.subject" content="Computational Biology/Bioinformatics">     
<META name="dc.subject" content="Computer Appl. in Life Sciences">     
<META name="dc.subject" content="Combinatorial Libraries">     
<META name="dc.subject" content="Algorithms">     
<META name="prism.issn" content="1471-2105">     
<META name="prism.doi" content="10.1186/s12859-016-1005-x">     
<META name="prism.publicationName" content="BMC Bioinformatics">     
<META name="prism.publicationDate" content="2016-04-12">     
<META name="prism.volume" content="17">     
<META name="prism.number" content="1">     
<META name="prism.section" content="OriginalPaper">     
<META name="prism.startingPage" content="1">     
<META name="prism.copyright" content="2016 Fu et al.">     
<META name="prism.rightsAgent" content="reprints@biomedcentral.com">     
<META name="prism.url" content="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x"> 
    
<META name="citation_journal_title" content="BMC Bioinformatics">     
<META name="citation_publisher" content="BioMed Central">     
<META name="citation_author" content="Gang Fu">     
<META name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, USA"> 
    
<META name="citation_author" content="Ying Ding">     
<META name="citation_author_institution" content="School of Informatics &amp; Computing, Indiana University, Bloomington, USA"> 
    
<META name="citation_author_institution" content="School of Information Management, Wuhan University, Wuchang, Wuhan, China"> 
    
<META name="citation_author" content="Abhik Seal">     
<META name="citation_author_institution" content="School of Informatics &amp; Computing, Indiana University, Bloomington, USA"> 
    
<META name="citation_author" content="Bin Chen">     
<META name="citation_author_institution" content="Department of Medicine, Stanford University, Stanford, USA"> 
    
<META name="citation_author" content="Yizhou Sun">     
<META name="citation_author_institution" content="College of Computer and Information Science, Northeastern University, Boston, USA"> 
    
<META name="citation_author" content="Evan Bolton">     
<META name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, USA"> 
    
<META name="citation_issn" content="1471-2105">     
<META name="citation_title" content="Predicting drug target interactions using meta-path-based semantic network analysis"> 
    
<META name="citation_volume" content="17">     
<META name="citation_issue" content="1">     
<META name="citation_date" content="2016-04-12">     
<META name="citation_firstpage" content="1">     
<META name="citation_doi" content="10.1186/s12859-016-1005-x">     
<META name="citation_pdf_url" content="http://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1005-x?site=http://bmcbioinformatics.biomedcentral.com"> 
    
<META name="citation_abstract_html_url" content="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x"> 
    
<META name="citation_fulltext_html_url" content="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x"> 
<META charset="UTF-8"> 
<META http-equiv="X-UA-Compatible" content="IE=edge"> 
<META name="viewport" content="width=device-width, initial-scale=1">     <LINK 
href="/static/images/apple-touch-icon-57x57.34ab60d5.png" rel="apple-touch-icon" 
sizes="57x57">     <LINK href="/static/images/apple-touch-icon-60x60.eb01a2ee.png" 
rel="apple-touch-icon" sizes="60x60">     <LINK href="/static/images/apple-touch-icon-72x72.23a129b9.png" 
rel="apple-touch-icon" sizes="72x72">     <LINK href="/static/images/apple-touch-icon-76x76.2c588c51.png" 
rel="apple-touch-icon" sizes="76x76">     <LINK href="/static/images/apple-touch-icon-114x114.5d69e424.png" 
rel="apple-touch-icon" sizes="114x114">     <LINK href="/static/images/apple-touch-icon-120x120.6a277ad3.png" 
rel="apple-touch-icon" sizes="120x120">     <LINK href="/static/images/apple-touch-icon-144x144.7e42d0e6.png" 
rel="apple-touch-icon" sizes="144x144">     <LINK href="/static/images/apple-touch-icon-152x152.977d3041.png" 
rel="apple-touch-icon" sizes="152x152">     <LINK href="/static/images/apple-touch-icon-180x180.395dbefe.png" 
rel="apple-touch-icon" sizes="180x180">     <LINK href="/static/images/android-chrome-192x192.b31a370e.png" 
rel="icon" type="image/png" sizes="192x192">     <LINK href="/static/images/favicon-16x16.c826aa68.png" 
rel="icon" type="image/png" sizes="16x16">     <LINK href="/static/images/favicon-32x32.44498c19.png" 
rel="icon" type="image/png" sizes="32x32">     <LINK href="/static/images/favicon-96x96.41f1ee94.png" 
rel="icon" type="image/png" sizes="96x96">     <LINK href="/static/images/favicon-194x194.764d0a99.png" 
rel="icon" type="image/png" sizes="194x194">     <LINK href="/static/images/favicons/biomedcentral/manifest.json" 
rel="manifest">     <LINK href="/static/images/favicon.dc10a51d.ico" rel="shortcut icon"> 
    
<META name="msapplication-TileColor" content="#da532c">     
<META name="msapplication-config" content="/static/images/favicons/biomedcentral/browserconfig.xml"> 
    
<META name="msapplication-TileImage" content="/static/images/mstile-144x144.6ec26f96.png"> 
    
<META name="theme-color" content="#ffffff"> <!--[if lt IE 9]>
<script src="/static/js/html5shiv-printshiv.min.8dc1d332.js"></script>
<![endif]--> 
<SCRIPT>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</SCRIPT>
 <LINK href="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/css.css" 
rel="stylesheet" type="text/css">     <LINK href="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/suis-biomedcentral.9b6dd529.css" 
rel="stylesheet" media="screen">     <LINK href="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/suis-biomedcentral-print.201cf5fb.css" 
rel="stylesheet" media="print"> <!-- Need this for analytics to work. Remove once analytics have been upgraded --> 
<SCRIPT>var _gaq = [];</SCRIPT>
     
<SCRIPT type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcbioinformatics.biomedcentral.com',
            siteWithPath: 'bmcbioinformatics.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcbioinformatics',
            
            
            publisherBrand: 'BioMed Central'
        };
    </SCRIPT>
     
<SCRIPT>
        (function (i, s, o, g, r, a, m) {
            i['GoogleAnalyticsObject'] = r;
            i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
            a = s.createElement(o),
                    m = s.getElementsByTagName(o)[0];
            a.async = 1;
            a.src = g;
            m.parentNode.insertBefore(a, m)
        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');

        var sampleRate = 20;

            
            ga('create', "UA-62065396-3",
                    {
                        cookieDomain: 'none',
                        allowLinker: true,
                        siteSpeedSampleRate: sampleRate
                    });
            ga("set", "anonymizeIp", true);
            ga('send', 'pageview');
            

        ga('create', "UA-54492316-9",
                {
                    cookieDomain: 'none',
                    allowLinker: true,
                    siteSpeedSampleRate: sampleRate,
                    name: 'global'
                });

        ga("global.set", "anonymizeIp", true);
        ga('global.send', 'pageview');
    </SCRIPT>
     <LINK href="//bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x" 
rel="canonical">         
<META content="http://bmcbioinformatics.biomedcentral.com" property="og:url">    
 
<META content="article" property="og:type">     
<META content="BMC Bioinformatics" property="og:title">     
<META content="<p><i>BMC Bioinformatics</i> is an open access journal publishing original peer-reviewed research articles in all aspects of the development, testing and novel application of computational and statistical methods for the modeling and analysis of all kinds of biological data, as well as other areas of computational biology.&nbsp;</p><p><i>BMC Bioinformatics</i> is part of the <i>BMC series</i> which publishes subject-specific journals focused on the needs of individual research communities across all areas of biology and medicine. We offer an efficient, fair and friendly peer review service, and are committed to publishing all sound science, provided that there is some advance in knowledge presented by the work.</p><p><i>BMC series - open, inclusive and trusted.</i></p>" 
property="og:description">     
<META content="https://images.springer.com/sgw/journals/medium/12859.jpg" 
property="og:image"> 
<META name="GENERATOR" content="MSHTML 11.00.9600.18212"></HEAD> 
<BODY class="fulltext">
<DIV class="Container" style="height: 0px;">
<DIV id="wt_refpoint"></DIV></DIV>
<P class="SkipToMain"><A class="SkipToMain_link" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#main-content">Skip 
to main content</A> </P>
<DIV id="top">
<DIV class="adsbox RealMediaAd Ad Ad--leaderboard" id="RealMediaLeaderboard" 
data-position="Top">
<DIV class="Ad_content">
<P class="Ad_label">Advertisement</P></DIV></DIV>
<SCRIPT type="text/javascript">
                var _kiq = _kiq || [];
                (function(){
                    setTimeout(function(){
                        var d = document, f = d.getElementsByTagName('script')[0], s = d.createElement('script'); s.type = 'text/javascript';
                        s.async = true; s.src = '//s3.amazonaws.com/ki.js/47412/9CC.js'; f.parentNode.insertBefore(s, f);
                    }, 1);
                    setTimeout(function(){
                        var d = document, f = d.getElementsByTagName('script')[0], s = d.createElement('script'); s.type = 'text/javascript';
                        s.async = true; s.src = '//s3.amazonaws.com/ki.js/47412/aC6.js'; f.parentNode.insertBefore(s, f);
                    }, 1);
                })();
            </SCRIPT>
             <HEADER class="Header" role="banner">
<DIV class="Header_inner"><A title="BioMed Central" class="Logo" id="header-logo" 
href="http://www.biomedcentral.com/"><IMG alt="Biomed Central logo" src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/biomed-central-logo_180x35.aea55d8f.png" 
srcset="/static/images/biomed-central-logo_180x35.85951291.svg">                 
    </A>                     
<DIV class="Header_right">
<DIV class="Toggles"><SPAN class="Toggles_item NavToggle" 
data-toggle="menu">Menu</SPAN>                             <SPAN class="Toggles_item SearchToggle" 
data-toggle="search"><SPAN class="SearchToggle_icon" 
aria-hidden="true"></SPAN></SPAN>                         </DIV>
<FORM class="Search Search--expandable" id="globalSearch" role="search" action="//www.biomedcentral.com/search" 
method="GET" data-event-label="" data-event-action="Submit search" 
data-event-category="Search and Results" data-track-event="submit">
<DIV class="Search_content"><LABEL class="Search_label " 
for="publisherSearch">Search</LABEL>         <INPUT name="query" title="Search BioMed Central" class="Search_input" id="publisherSearch" role="textbox" type="text" autocomplete="off"><BUTTON 
class="Search_btn" id="search" type="submit"><SPAN 
class="u-srOnly">Search</SPAN>             <SPAN class="Search_icon"></SPAN>     
    </BUTTON>         <INPUT name="searchType" type="hidden" value="publisherSearch"> 
    </DIV></FORM></DIV><NAV class="GlobalNav js-fadeToggle" id="globalNavigation" 
role="navigation" aria-label="publisher">
<H2 class="u-srOnly">Publisher main menu</H2>
<UL class="GlobalNav_list" id="publisherNavigation">
  <LI><A href="http://www.biomedcentral.com/journals">Journals</A></LI>
  <LI><A href="http://www.biomedcentral.com/submissions">Submissions</A></LI>
  <LI><A href="http://www.biomedcentral.com/publishing-services">Publishing 
  services</A></LI>
  <LI><A href="http://www.biomedcentral.com/about">About</A></LI></UL>
<P class="GlobalNav_account" id="loginLink"><A href="http://www.biomedcentral.com/login-redirect-info" 
data-event-action="Click Account" data-event-category="Account" 
data-track-event="click">My Account</A></P></NAV>
<DIV class="placeholder" id="membership-message-loader-desktop" 
data-placeholder="/placeholder/v1/membership/message"></DIV></DIV>
<H3 class="u-srOnly">Follow BioMed Central</H3>
<UL class="Follow Follow--stacked">
  <LI class="Follow_item"><A class="Follow_link" href="https://twitter.com/biomedcentral" 
  data-event-label="Twitter" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Follow_icon Follow_icon--twitter"></SPAN><SPAN 
  class="u-srOnly">Twitter</SPAN>                         </A></LI>
  <LI class="Follow_item"><A class="Follow_link" href="https://www.facebook.com/BioMedCentral" 
  data-event-label="Facebook" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Follow_icon Follow_icon--facebook"></SPAN><SPAN 
  class="u-srOnly">Facebook</SPAN>                         
</A></LI></UL></HEADER><main role="main"><HEADER class="identity" role="banner">
<DIV class="identity__cover identity__cover--networks-aqua">
<DIV class="identity__container">
<H2 class="identity__title" id="journalTitle"><A class="identity__title-link" 
href="http://bmcbioinformatics.biomedcentral.com/">BMC 
Bioinformatics</A></H2><NAV class="identity__nav" role="navigation" aria-label="journal">
<H3 class="u-srOnly">BMC Bioinformatics main menu</H3>
<UL class="identity__nav-list Nav" id="siteNavigation" role="menu">
  <LI class="identity__nav-item" role="menuitem"><A class="identity__nav-link " 
  href="http://bmcbioinformatics.biomedcentral.com/about"><SPAN class="identity__nav-text"><SPAN>About</SPAN></SPAN></A></LI>
  <LI class="identity__nav-item" role="menuitem"><A class="identity__nav-link isActive" 
  href="http://bmcbioinformatics.biomedcentral.com/articles"><SPAN class="identity__nav-text"><SPAN>Articles</SPAN></SPAN></A></LI>
  <LI class="identity__nav-item" role="menuitem"><A class="identity__nav-link " 
  href="http://bmcbioinformatics.biomedcentral.com/submission-guidelines"><SPAN 
  class="identity__nav-text"><SPAN>Submission 
Guidelines</SPAN></SPAN></A></LI></UL></NAV></DIV></DIV></HEADER>
<DIV class="Main Container" id="main-content">
<DIV class="Main_content" id="Test-ImgSrc">
<DIV class="FulltextWrapper">
<DIV class="Categories" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<DIV class="ArticleCategory" xmlns="http://www.w3.org/1999/xhtml">Research 
article</DIV>
<DIV class="OpenAccessLabel">Open Access</DIV></DIV></DIV>
<DIV class="FulltextWrapper">
<DIV class="MainTitleSection" xmlns:meta="http://www.springer.com/app/meta" 
xmlns="">
<H1 class="ArticleTitle" lang="en" 
xmlns="http://www.w3.org/1999/xhtml">Predicting drug target interactions using 
meta-path-based semantic network analysis</H1></DIV>
<DIV class="AuthorNames" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<UL class="u-listReset">
  <LI class="Author" data-jumpto="Aff"><SPAN class="AuthorName" xmlns="http://www.w3.org/1999/xhtml">Gang&nbsp;Fu</SPAN><SUP><A 
  class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff1">1</A></SUP><A 
  title="Email author" class="EmailAuthor" href="mailto:gang.fu@nih.gov"><SPAN 
  class="ContactIcon" aria-hidden="true"></SPAN><SPAN class="u-srOnly">Email 
  author</SPAN></A>, </LI>
  <LI class="Author" data-jumpto="Aff"><SPAN class="AuthorName" xmlns="http://www.w3.org/1999/xhtml">Ying&nbsp;Ding</SPAN><SUP><A 
  class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff2">2</A>, 
  <A class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff3">3</A></SUP>, 
  </LI>
  <LI class="Author" data-jumpto="Aff"><SPAN class="AuthorName" xmlns="http://www.w3.org/1999/xhtml">Abhik&nbsp;Seal</SPAN><SUP><A 
  class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff2">2</A></SUP>, 
  </LI>
  <LI class="Author" data-jumpto="Aff"><SPAN class="AuthorName" xmlns="http://www.w3.org/1999/xhtml">Bin&nbsp;Chen</SPAN><SUP><A 
  class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff4">4</A></SUP>, 
  </LI>
  <LI class="Author" data-jumpto="Aff"><SPAN class="AuthorName" xmlns="http://www.w3.org/1999/xhtml">Yizhou&nbsp;Sun</SPAN><SUP><A 
  class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff5">5</A></SUP> 
  and </LI>
  <LI class="Author" data-jumpto="Aff"><SPAN class="AuthorName" xmlns="http://www.w3.org/1999/xhtml">Evan&nbsp;Bolton</SPAN><SUP><A 
  class="AffiliationID" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Aff1">1</A></SUP></LI></UL></DIV>
<DIV class="ArticleContextInformation" xmlns:meta="http://www.springer.com/app/meta" 
xmlns="">
<DIV class="ContextInformation">
<DIV class="ContextInformationJournalTitles"><SPAN class="ArticleCitation"><SPAN 
class="JournalTitle" xmlns="http://www.w3.org/1999/xhtml">BMC 
Bioinformatics</SPAN><SPAN class="JournalSubTitle" xmlns="http://www.w3.org/1999/xhtml">BMC 
series – open, inclusive and trusted</SPAN><SPAN 
class="ArticleCitation_Year">2016</SPAN><SPAN 
class="ArticleCitation_Volume"><STRONG>17</STRONG>:160</SPAN></SPAN></DIV>
<P class="ArticleDOI"><STRONG>DOI: </STRONG>10.1186/s12859-016-1005-x</P>
<P class="Copyright">© &nbsp;Fu et al.&nbsp;2016</P></DIV></DIV>
<DIV class="ArticleHistory">
<P class="History HistoryReceived"><STRONG>Received: 
</STRONG>30&nbsp;April&nbsp;2015</P>
<P class="History HistoryAccepted"><STRONG>Accepted: 
</STRONG>31&nbsp;March&nbsp;2016</P>
<P class="History HistoryOnlineDate"><STRONG>Published: 
</STRONG>12&nbsp;April&nbsp;2016</P></DIV></DIV>
<DIV class="FulltextWrapper"><SECTION class="Abstract" id="Abs1" lang="en" 
xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Abstract</H2>
<DIV class="js-CollapseSection">
<DIV class="AbstractSection" id="ASec1" xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Background</H3>
<P class="Para" id="Par1">In the context of drug discovery, drug target 
interactions (DTIs) can be predicted based on observed topological features of a 
semantic network across the chemical and biological space. In a semantic 
network, the types of the nodes and links are different. In order to take into 
account the heterogeneity of the semantic network, meta-path-based topological 
patterns were investigated for link prediction.</P></DIV>
<DIV class="AbstractSection" id="ASec2" xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Results</H3>
<P class="Para" id="Par2">Supervised machine learning models were constructed 
based on meta-path topological features of an enriched semantic network, which 
was derived from Chem2Bio2RDF, and was expanded by adding compound and protein 
similarity neighboring links obtained from the PubChem databases. The additional 
semantic links significantly improved the predictive performance of the 
supervised learning models. The binary classification model built upon the 
enriched feature space using the Random Forest algorithm significantly 
outperformed an existing semantic link prediction algorithm, Semantic Link 
Association Prediction (SLAP), to predict unknown links between compounds and 
protein targets in an evolving network. In addition to link prediction, Random 
Forest also has an intrinsic feature ranking algorithm, which can be used to 
select the important topological features that contribute to link 
prediction.</P></DIV>
<DIV class="AbstractSection" id="ASec3" xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Conclusions</H3>
<P class="Para" id="Par3">The proposed framework has been demonstrated as a 
powerful alternative to SLAP in order to predict DTIs using the semantic network 
that integrates chemical, pharmacological, genomic, biological, functional, and 
biomedical information into a unified framework. It offers the flexibility to 
enrich the feature space by using different normalization processes on the 
topological features, and it can perform model construction and feature 
selection at the same time.</P></DIV>
<DIV class="KeywordGroup" lang="en" xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Keywords</H3><SPAN class="Keyword">Semantic network 
analysis</SPAN>               <SPAN class="Keyword">Link prediction</SPAN>       
        <SPAN class="Keyword">Meta-path topological feature</SPAN>               
<SPAN class="Keyword">Machine learning</SPAN>               <SPAN class="Keyword">Random 
forest</SPAN>             </DIV></DIV></SECTION><ASIDE class="ArticleNote ArticleNoteESMHint">
<H3 class="Heading" xmlns:meta="http://www.springer.com/app/meta" 
xmlns="">Electronic supplementary material</H3>
<P class="SimplePara" xmlns:meta="http://www.springer.com/app/meta" xmlns="">The 
online version of this article (doi:<SPAN class="ExternalRef" xmlns="http://www.w3.org/1999/xhtml"><A 
href="http://dx.doi.org/10.1186/s12859-016-1005-x"><SPAN 
class="RefSource">10.​1186/​s12859-016-1005-x</SPAN></A></SPAN>) contains 
supplementary material, which is available to authorized users.</P></ASIDE>      
       <SECTION class="Section1 RenderAsSection1 SectionTypeIntroduction" id="Sec1" 
xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Background</H2>
<DIV class="js-CollapseSection">
<P class="Para" id="Par13" xmlns="http://www.w3.org/1999/xhtml">Chemogenomics 
[<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR1">1</A></SPAN>, 
<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR2">2</A></SPAN>] 
and chemical systems biology [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR3">3</A></SPAN>, 
<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR4">4</A></SPAN>] 
aim to accelerate drug discovery inexpensively through <EM class="EmphasisTypeItalic" 
xmlns="">in silico</EM> predictions, based on a network with enriched 
drug-target-disease relationships [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR5">5</A></SPAN>]. 
Integrated chemical and biological networks can be used to hypothesize new 
clinical indications for approved drugs with desired safety profiles, and to 
propose new combination therapy design [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR6">6</A></SPAN>, 
<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR7">7</A></SPAN>]. 
Drug-target interaction networks can also be utilized to interpret clinical side 
effects by revealing modes of drug actions [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR8">8</A></SPAN>]. 
Semantic standards and technologies facilitate seamless data integration across 
multiple domains, and enable the construction of a heterogeneous network 
consisting of various biological entities of different types, such as compounds, 
proteins, and genes [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR9">9</A></SPAN>]. 
Several semantically linked datasets, such as PubChemRDF [<SPAN class="CitationRef"><A 
href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR10">10</A></SPAN>], 
Chem2Bio2Rdf [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR11">11</A></SPAN>], 
Bio2RDF [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR12">12</A></SPAN>], 
Open PHACTS [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR13">13</A></SPAN>], 
and ChEMBL RDF [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR14">14</A></SPAN>], 
have been published to promote large-scale data mining in drug discovery. A 
statistical model, called Semantic Link Association Prediction (SLAP), has been 
applied to Chem2Bio2RDF to predict direct links between compounds and proteins 
based on their indirect links or paths with other biological objects, such as 
substructures, diseases, side effects, and pathways [<SPAN 
class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR15">15</A></SPAN>]. 
It has been demonstrated that SLAPas a novel and validated approach to predict 
drug-target interactions (DTIs) outperformed existing alternatives.</P>
<P class="Para" id="Par14" xmlns="http://www.w3.org/1999/xhtml">Predicting DTI 
is equivalent to link prediction, which is a fundamental problem and 
long-standing challenge in complex network analysis [<SPAN 
class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR16">16</A></SPAN>]. 
In social networks, topological proximity, measured based on observed network 
data, can be used to suggest future interactions between individuals [<SPAN 
class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR17">17</A></SPAN>]. 
In the context of drug discovery, biological networks can be similarly leveraged 
to identify potential associations between compounds and protein targets. 
Typical network-based DTI predictions are often based on similarity profiles 
calculated from common neighbors or direct connections, and are usually limited 
to bipartite networks [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR18">18</A></SPAN>–<SPAN 
class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR21">21</A></SPAN>]. 
However, most similarity-based link prediction algorithms designed for 
homogeneous networks cannot take into account the heterogeneous types and 
relations defined in semantic networks; furthermore, it is fairly challenging to 
consider the long paths connecting two end nodes (indirect connections), which 
can significantly increase large volumes of randomness in the connectivity. 
Therefore, we incorporated meta-path topological features [<SPAN class="CitationRef"><A 
href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR22">22</A></SPAN>] 
for link prediction. A meta-path is a composite relation, denoting a sequence of 
adjacent links between any two objects in a heterogeneous network. Adjacent 
links are defined with distinct semantics, so different combinations of adjacent 
links in sequences contribute distinguishably for link prediction. It has been 
proven that meta-path-based similarity can improve the performance of 
information retrieval in heterogeneous information networks [<SPAN class="CitationRef"><A 
href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR23">23</A></SPAN>].</P>
<P class="Para" id="Par15" xmlns="http://www.w3.org/1999/xhtml">A meta-path 
defines a certain type of paths linking the starting and ending objects. The 
total number of paths belonging to a specific meta-path is animportant 
topological feature to evaluate the strength of associations between starting 
and ending objects, which is often called path count. For instance, a compound 
and a protein target can be connected through multiple paths of different types: 
(A) compound <SPAN class="InlineEquation" id="IEq53">\( \overset{similar\ 
to}{\to } \)</SPAN> compound <SPAN class="InlineEquation" id="IEq54">\( 
\overset{binds\ to}{\to } \)</SPAN> protein; (B) compound <SPAN class="InlineEquation" 
id="IEq55">\( \overset{binds\ to}{\to } \)</SPAN> protein <SPAN class="InlineEquation" 
id="IEq56">\( \overset{binds\ to}{\to } \)</SPAN> compound <SPAN class="InlineEquation" 
id="IEq57">\( \overset{binds\ to}{\to } \)</SPAN> protein; and (C) compound 
<SPAN class="InlineEquation" id="IEq58">\( \overset{has\  part}{\to } \)</SPAN> 
substructure <SPAN class="InlineEquation" id="IEq59">\( \overset{part\  of}{\to 
} \)</SPAN> compound <SPAN class="InlineEquation" id="IEq60">\( \overset{binds\ 
to}{\to } \)</SPAN> protein <SPAN class="InlineEquation" id="IEq61">\( 
\overset{similar\ to}{\to } \)</SPAN> protein. Three meta-paths connect the 
starting compound to the ending protein: meta-path (A) indicates that the 
compound most likely binds to a protein to which another structurally similar 
compound binds; meta-path (B) shows that two compounds sharing an observed 
protein target may share another protein target as well; meta-path (C) specifies 
that two compounds sharing a common substructure may bind to two different 
protein targets that have similar protein sequences. SLAP employs a statistical 
model to evaluate the importance of each meta-path in link prediction, which is 
evaluated individually based on the distribution of its connectivity property 
over a set of randomly sampled drug-target pairs. Several meta-paths are 
selected according to their statistical significances, and the aggregated 
connectivity properties of the selected meta-paths are used to predict DTI.</P>
<P class="Para" id="Par16" xmlns="http://www.w3.org/1999/xhtml">The present work 
provides an alternative DTI approach to SLAP. Rather than using a statistical 
model to study the significance of meta-path topological features, we propose a 
framework to take advantage of machine learning algorithms, including Random 
Forest (RF) and Support Vector Machine (SVM), to construct binary classification 
models to predict DTI. A more complete drug-target connectivity map can be 
constructed using the predicted links. By using machine learning models, feature 
importance (i.e., the contributions of different meta-paths to the link 
prediction) can be calculated at the same time as the classification models are 
built. Additionally, SLAP only considers path counts as a topological feature; 
whereas our approach can apply different kinds of normalization processes to 
path counts, including random walk, normalized path count, and symmetric random 
walk [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR23">23</A></SPAN>] 
to further enrich the topological feature space. In order to compare our 
approach with SLAP, we have carried out link prediction experiments on a 
semantic network, called Chem2Bio2Rdf, which focuses on drug candidates and 
their biological annotations. Although the proposed approach was just used to 
construct a more complete drug-target connectivity map in the present study, it 
can be generalized as a framework to leverage machine learning algorithms to 
study the topological features of the heterogeneous network for link prediction. 
Structural similarity links between compounds and sequence similarity links 
between proteins were added to expand the semantic network. The usefulness of 
similarity neighboring links from PubChem resources [<SPAN 
class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR24">24</A></SPAN>] 
is examined in the context of semantic link 
prediction.</P></DIV></SECTION><SECTION class="Section1 RenderAsSection1" id="Sec2" 
xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Methods</H2>
<DIV class="js-CollapseSection"><SECTION class="Section2 RenderAsSection2" id="Sec3">
<H3 class="Heading">Semantic network</H3>
<DIV class="Para" id="Par17" xmlns="http://www.w3.org/1999/xhtml">In the 
Chem2Bio2RDF semantic network, nine distinct semantic types are presented, 
including compounds, proteins, adverse side effects, Gene Ontology (GO) 
annotations, ChEBI types, substructures, tissues, biological pathways, and 
diseases; ten different semantic links are incorporated, including links from 
compounds to ChEBI types, from compounds to proteins, from compounds to 
substructures, from adverse side effects to compounds, from diseases to 
compounds, from proteins to proteins (referring to protein-protein 
interactions), from proteins to GO annotations, from diseases to proteins, from 
pathways to proteins, and from tissues to proteins. In order to enhance link 
prediction performance, we enriched the linked dataset by adding two more 
semantic links: compound neighboring links based on 2D structural similarity, 
and protein neighboring links, based on sequence similarity. The similarity 
neighboring links were obtained from PubChem databases [<SPAN class="CitationRef"><A 
href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR25">25</A></SPAN>, 
<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR26">26</A></SPAN>]. 
A total of twelve adjacency matrixes were computed based on the semantic links 
between any two objects. The elements of the adjacency matrixes have two values: 
‘0,’ indicating unobserved links, and ‘1,’ indicating observed links. The 
semantics and statistics of adjacency matrixes were enumerated in 
Table&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Tab1">1</A></SPAN>; 
these were used to calculate the meta-path-based topological features. It is 
noteworthy that all the semantic links in the Chem2Bio2RDF dataset are 
reversible, and the adjacency matrix for the reverse semantic links can be 
obtained through a transpose of the original adjacency matrix.<FIGURE class="FigureTable" 
id="Tab1" xmlns=""><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Table 
1</SPAN>                         
<P class="SimplePara" xmlns="">The semantics and statistics of adjacency 
matrixes</P></DIV></FIGCAPTION>
<DIV class="Table">
<TABLE border="1" xmlns="http://www.w3.org/1999/xhtml">
  <COLGROUP>
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left"></COLGROUP>
  <THEAD>
  <TR class="header">
    <TH align="left">
      <P class="SimplePara" xmlns="">Index</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Semantics</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">From</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Number of Rows</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">To</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Number of Colums</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Count<SUP xmlns="http://www.w3.org/1999/xhtml"><EM 
      class="EmphasisTypeItalic" xmlns="">a</EM>                               
      </SUP>                                           </P></TH></TR></THEAD>
  <TBODY>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A1</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">has ChEBI type</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">ChEBI type</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">2777</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">14633</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A2</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">binds to</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">528831</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A3</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">has part</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">substructure</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">290</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">6127</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A4</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">induced by</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">adverse side effect</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">1051</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">9004</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A5</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">treated by</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">disease</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">1284</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">927</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A6</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">interacts with</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">72773</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A7</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">has GO annotation</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">GO annotation</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">9710</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">89688</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A8</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">caused by</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">disease</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">1284</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">2676</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A9</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">has participants</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">pathway</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">192</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">10796</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A10</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">expresses</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">tissue</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">507</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">9905</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A11</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">similar to</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">compound</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">258030</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">6184722</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">A12</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">similar to</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">protein</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">22056</P></TD>
    <TD align="right">
      <P class="SimplePara" xmlns="">261158</P></TD></TR></TBODY></TABLE></DIV>
<DIV class="TableFooter" xmlns="http://www.w3.org/1999/xhtml">
<P class="SimplePara" xmlns=""><SUP xmlns="http://www.w3.org/1999/xhtml"><EM 
class="EmphasisTypeItalic" xmlns="">a</EM>                         </SUP>The 
number of non-zero elements in adjacency 
matrix</P></DIV></FIGURE></DIV></SECTION><SECTION class="Section2 RenderAsSection2" 
id="Sec4">
<H3 class="Heading">Meta-path-based topological features</H3>
<DIV class="Para" id="Par18" xmlns="http://www.w3.org/1999/xhtml">The meta-path 
topological features were encoded in commuting matrixes, calculated by 
multiplying several adjacency matrixes. To predict the links from compounds to 
proteins, we exhaustively enumerate all the possible meta-paths, yielding a 
total of 51 meta-paths. Each commuting matrix represents a certain type of 
meta-path of a given length. The length of the meta-paths equals the number of 
multiplied adjacency matrixes. Out of 51 commuting matrixes, 4 meta-paths are of 
length 2; 11 meta-paths are of length 3; and 36 meta-paths are of length 4. The 
meta-paths with length greater than 4 are considered to be too long to make a 
significant contribution to link prediction. The elements in the commuting 
matrix indicate the number of path instances linking compounds to proteins, and 
have non-negative integer values. The semantics and statistics of commuting 
matrixes were enumerated in Table&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Tab2">2</A></SPAN>. 
For instance, the commuting matrix C15 represents a meta-path: compound <SPAN 
class="InlineEquation" id="IEq62">\( \overset{similar\ to}{\to } \)</SPAN> 
compound <SPAN class="InlineEquation" id="IEq63">\( \overset{binds\ to}{\to } 
\)</SPAN> protein <SPAN class="InlineEquation" id="IEq64">\( \overset{similar\ 
to}{\to } \)</SPAN> protein, which was calculated by multiplying three adjacency 
matrixes: A2, A11, and A12 (Fig.&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Fig1">1</A></SPAN>). 
All of the matrix multiplications were carried out using the Armadillo C++ 
linear algebra library [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR27">27</A></SPAN>], 
and all of the adjacency and commuting matrixes were encoded as sparse matrixes 
to reduce memory consumption.<FIGURE class="FigureTable" id="Tab2" 
xmlns=""><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Table 
2</SPAN>                         
<P class="SimplePara" xmlns="">The semantics and statistics of commuting 
matrixes</P></DIV></FIGCAPTION>
<DIV class="Table">
<TABLE border="1" xmlns="http://www.w3.org/1999/xhtml">
  <COLGROUP>
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left"></COLGROUP>
  <THEAD>
  <TR class="header">
    <TH align="left">
      <P class="SimplePara" xmlns="">Index</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Semantics</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Count<SUP xmlns="http://www.w3.org/1999/xhtml"><EM 
      class="EmphasisTypeItalic" xmlns="">a</EM>                               
      </SUP>                                           </P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">Max<SUP xmlns="http://www.w3.org/1999/xhtml"><EM 
      class="EmphasisTypeItalic" xmlns="">b</EM>                               
      </SUP>                                           </P></TH></TR></THEAD>
  <TBODY>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C1</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq1" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{similar\;to}{\to}\;\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1995778</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">395</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C2</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq2" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{interacts\; 
      with}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">4878633</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">20</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C3</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq3" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{similar\ 
      to}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">30665527</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">84</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C4</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq4" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{treats}{\to}\;\mathrm{disease}\overset{caused\;by}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">6178</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">3</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C5</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq5" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{similar\;to}{\to}\;\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">15086309</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">934</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C6</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq6" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{similar\;to}{\to}\;\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\ 
      to}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">49226573</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1163</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C7</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq7" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{binds\;to}{\to}\mathrm{compound}\overset{binds\ 
      to}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">126339670</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">30400</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C8</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq8" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ \overset{has\; 
      part}{\to}\;\mathrm{substructure}\overset{part\; 
      of}{\to}\mathrm{compound}\overset{binds\ to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">922056</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">202</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C9</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq9" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ \overset{has\; 
      type}{\to}\;\mathrm{ChEBI}\ \mathrm{type}\overset{type\  
      of}{\to}\mathrm{compound}\overset{binds\ to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">709802</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">324</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C10</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq10" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{induces}{\to}\;\mathrm{adverse}\ \mathrm{side}\ 
      \mathrm{effect}\overset{induced\kern0.5em 
      by}{\to}\mathrm{compound}\overset{binds\ to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">420616</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">194</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C11</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq11" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{treats}{\to}\;\mathrm{disease}\overset{treated\;by}{\to}\mathrm{compound}\overset{binds\ 
      to}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">68479</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">25</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C12</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq12" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{has\; 
      annotation}{\to}\mathrm{GO}\ \mathrm{annotation}\overset{annotation\  
      of}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">316095950</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">335</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C13</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq13" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{participates\; 
      in}{\to}\mathrm{pathway}\overset{has\  participants}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">82834409</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">328</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C14</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq14" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{expressed\; 
      in}{\to}\mathrm{tissue}\overset{expresses}{\to}\mathrm{protein} \)</SPAN>  
                                 </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">53586080</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">76</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C15</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq15" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{causes}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1360337</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">10</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C16</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq16" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{binds\;to}{\to}\mathrm{compound}\overset{binds\ 
      to}{\to}\mathrm{protein}\overset{interact\  with}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">522513250</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">142290</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C17</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq17" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\;\mathrm{protein}\overset{binds\;to}{\to}\mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">12963831</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">498</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C18</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq18" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\mathrm{protein}\ 
      \overset{binds\;to}{\to}\mathrm{compound}\ 
      \overset{similar\;to}{\to}\mathrm{compound}\ 
      \overset{binds\;to}{\to}\mathrm{protein} \)</SPAN>                         
          </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">201052081</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">777576</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C19</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq19" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ 
      \overset{binds\;to}{\to}\mathrm{protein}\ 
      \overset{binds\;to}{\to}\mathrm{compound}\ 
      \overset{binds\;to}{\to}\mathrm{protein}\ 
      \overset{similar\;to}{\to}\mathrm{protein} \)</SPAN>                       
            </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">356122463</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">445332</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C20</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq20" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ \overset{type\; 
      of}{\to}\mathrm{ChEBI}\ \mathrm{type}\ \overset{type\; 
      of}{\to}\mathrm{compound}\ 
      \overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\  
      with}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2333739</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2711</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C21</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq21" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\ \overset{type\; 
      of}{\to}\mathrm{ChEBI}\ \mathrm{type}\ \overset{type\; 
      of}{\to}\mathrm{compound}\overset{treats}{\to }\ 
      \mathrm{disease}\overset{caused\;by}{\to}\mathrm{protein} \)</SPAN>        
                           </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">190923</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">194</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C22</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq22" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{type\; 
      of}{\to}\mathrm{ChEBI}\ \mathrm{type}\overset{type\; 
      of}{\to}\mathrm{compound}\ \overset{binds\ to}{\to}\mathrm{protein}\ 
      \overset{similar\;to}{\to}\mathrm{protein} \)</SPAN>                       
            </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1463743</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">8639</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C23</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq23" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{type\; 
      of}{\to}\mathrm{ChEBI}\ \mathrm{type}\overset{type\  
      of}{\to}\mathrm{compound}\overset{similar\ 
      to}{\to}\mathrm{compound}\overset{binds\ to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">922257</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">8402</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C24</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq24" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{treated\ 
      by}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">371971</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">162</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C25</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq25" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{treated\ 
      by}{\to}\mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">38708</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">91</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C26</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq26" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{treated\ 
      by}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">493976</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">400</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C27</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq27" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{treated\ 
      by}{\to}\mathrm{compound}\overset{similar\;to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">106013</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">710</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C28</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq28" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{induces}{\to}\mathrm{adverse}\ \mathrm{side}\ 
      \mathrm{effect}\overset{induced\ 
      by}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1766464</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1622</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C29</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq29" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{induces}{\to}\mathrm{adverse}\ \mathrm{side}\ 
      \mathrm{effect}\overset{induced\ by}{\to}\mathrm{compound}\ 
      \overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">168841</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">106</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C30</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq30" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{induces}{\to}\mathrm{adverse}\ \mathrm{side}\ 
      \mathrm{effect}\overset{induced\ 
      by}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1193429</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">5571</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C31</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq31" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{induces}{\to}\mathrm{adverse}\ \mathrm{side}\ 
      \mathrm{effect}\overset{induced\ 
      by}{\to}\mathrm{compound}\overset{similar\;to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">765725</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2744</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C32</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq32" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{has\  
      part}{\to}\mathrm{substructure}\overset{part\  
      of}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">3465967</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">902</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C33</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq33" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{has\  
      part}{\to}\mathrm{substructure}\overset{part\  
      of}{\to}\mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">355993</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">96</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C34</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq34" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{has\  
      part}{\to}\mathrm{substructure}\overset{part\  
      of}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2175094</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2753</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C35</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq35" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{has\  
      part}{\to}\mathrm{substructure}\overset{part\  
      of}{\to}\mathrm{compound}\overset{similar\;to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1206786</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">12048</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C36</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq36" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein}\ \overset{has\  annotation}{\to}\mathrm{GO}\ 
      \mathrm{annotation}\overset{annotation\  of}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1064451402</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1929</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C37</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq37" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein}\overset{has\  annotation}{\to}\mathrm{GO}\ 
      \mathrm{annotation}\overset{annotation\  of}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2280505</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">136</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C38</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq38" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein}\overset{has\ 
       annotation}{\to}\mathrm{GO}\ \mathrm{annotation}\overset{annotation\  
      of}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">1480055439</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">50667</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C39</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq39" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{similar\ 
      to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{has\ 
       annotation}{\to}\mathrm{GO}\ \mathrm{annotation}\overset{annotation\  
      of}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">582316693</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">7765</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C40</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq40" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein}\overset{participates\  
      in}{\to}\mathrm{pathway}\overset{has\kern0.5em  
      participants}{\to}\mathrm{protein} \)</SPAN>                             
      </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">246398750</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2989</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C41</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq41" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein}\overset{participates\  
      in}{\to}\mathrm{pathway}\overset{has\  participants}{\to}\mathrm{protein} 
      \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">486267</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">183</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C42</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq42" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein}\overset{\ 
       participates\  in}{\to}\mathrm{pathway}\overset{\  has\kern0.5em  
      participants}{\to}\mathrm{protein} \)</SPAN>                             
      </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">358346529</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">73327</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C43</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq43" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{similar\ 
      to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{\ 
       participates\kern0.5em  in}{\to}\mathrm{pathway}\overset{\  has\kern0.5em 
       participants}{\to}\mathrm{protein} \)</SPAN>                             
      </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">149299008</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">7543</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C44</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq44" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein}\overset{causes}{\to}\mathrm{disease}\ 
      \overset{caused\kern0.5em by}{\to}\mathrm{protein} \)</SPAN>               
                    </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">7603639</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">44</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C45</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq45" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein}\overset{causes}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">27193</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">63</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C46</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq46" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein}\overset{causes}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">26747896</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">802</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C47</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq47" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{similar\;to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{causes}{\to}\mathrm{disease}\overset{caused\ 
      by}{\to}\mathrm{protein} \)</SPAN>                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">4159753</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">313</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C48</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq48" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{interacts\ 
       with}{\to}\mathrm{protein}\overset{expressed\  
      in}{\to}\mathrm{tissue}\overset{expresses}{\to}\mathrm{protein} \)</SPAN>  
                                 </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">222288200</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">453</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C49</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq49" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{treats}{\to}\mathrm{disease}\overset{caused\;by}{\to}\mathrm{protein}\overset{expressed\ 
       in}{\to}\mathrm{tissue}\overset{expresses}{\to}\mathrm{protein} \)</SPAN> 
                                  </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">300620</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">27</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C50</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq50" 
      xmlns="http://www.w3.org/1999/xhtml">\( \mathrm{compound}\overset{binds\ 
      to}{\to}\mathrm{protein}\overset{similar\;to}{\to}\mathrm{protein}\overset{expressed\ 
       in}{\to}\mathrm{tissue}\overset{expresses}{\to}\mathrm{protein} \)</SPAN> 
                                  </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">431134094</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">5974</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">C51</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns=""><SPAN class="InlineEquation" id="IEq51" 
      xmlns="http://www.w3.org/1999/xhtml">\( 
      \mathrm{compound}\overset{similar\;to}{\to}\mathrm{compound}\overset{binds\;to}{\to}\mathrm{protein}\overset{expressed\ 
       in}{\to}\mathrm{tissue}\overset{expresses}{\to}\mathrm{protein} \)</SPAN> 
                                  </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">117576353</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">2031</P></TD></TR></TBODY></TABLE></DIV>
<DIV class="TableFooter" xmlns="http://www.w3.org/1999/xhtml">
<P class="SimplePara" xmlns=""><SUP xmlns="http://www.w3.org/1999/xhtml"><EM 
class="EmphasisTypeItalic" xmlns="">a</EM>                         </SUP> The 
number of non-zero elements in commuting matrix; <SUP xmlns="http://www.w3.org/1999/xhtml"><EM 
class="EmphasisTypeItalic" xmlns="">b</EM>                         </SUP> the 
max value of element in commuting matrix.</P></DIV></FIGURE><FIGURE class="Figure" 
id="Fig1" xmlns="">
<DIV class="MediaObject" id="MO1" xmlns="http://www.w3.org/1999/xhtml"><IMG alt="https://static-content.springer.com/image/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_Fig1_HTML.gif" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/12859_2016_1005_Fig1_HTML.gif"> 
                    </DIV><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Fig. 
1</SPAN>                         
<P class="SimplePara" xmlns="">Schematic representation of calculations of 
commuting matrix C15 through multiplying A2, A11, and 
A12</P></DIV></FIGCAPTION></FIGURE></DIV>
<P class="Para" id="Par19" xmlns="http://www.w3.org/1999/xhtml">Two measures of 
topological features were calculated. Path count (<EM class="EmphasisTypeItalic" 
xmlns="">PC</EM>                            <SUB><EM class="EmphasisTypeItalic" 
xmlns="">i</EM>,<EM class="EmphasisTypeItalic" xmlns="">j</EM>                   
         </SUB>) measures the number of path instances between nodes <EM class="EmphasisTypeItalic" 
xmlns="">i</EM> and <EM class="EmphasisTypeItalic" xmlns="">j</EM>, which 
corresponds to the value of element in the commuting matrix. We also applied 
Random Walk (RW) as a normalization process to the number of path instances, 
based on the overall connectivity of the network. RW was calculated as <SPAN 
class="InlineEquation" id="IEq198">\( \raisebox{1ex}{$P{C}_{i,j}$}\!\left/ 
\!\raisebox{-1ex}{$P{C}_{i,\bullet }$}\right. \)</SPAN>, where <EM class="EmphasisTypeItalic" 
xmlns="">PC</EM>                            <SUB><EM class="EmphasisTypeItalic" 
xmlns="">i</EM>,•</SUB> are row-wise summations.</P></SECTION><SECTION class="Section2 RenderAsSection2" 
id="Sec5">
<H3 class="Heading">Machine learning dataset</H3>
<P class="Para" id="Par20" xmlns="http://www.w3.org/1999/xhtml">In order to 
build supervised learning models, both positive and negative labels are 
required. We treated observed links between compounds and protein targets as 
positive labels. A total of 5,387 positively labeled links from Drugbank were 
collected, which were used to evaluate the predictive performance of the SLAP 
algorithm [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR15">15</A></SPAN>]. 
The unobserved links in the dataset can be either spurious links or potential 
future links. In order to obtain experimental evidence for the negative labels, 
we surveyed the PubChem BioAssay database [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR28">28</A></SPAN>]: 
if the experimental bioactivity value is greater than 10&nbsp;μM, the link of a 
compound protein pair is negatively labeled. Accordingly, we obtained 26,682 
negative labels out of over 5.6 billion unobserved links between compounds and 
proteins in the Chem2Bio2RDF semantic network. In order to assess predictive 
performance without prior knowledge, the positively labeled links were removed 
from Chem2Bio2RDF when the meta-path-based topological features were calculated. 
The positively and negatively labeled links were combined and randomly split 
into training and test sets by a ratio of 2:1. In the training set, there are 
3,591 positively labeled links and 17,788 negatively labeled links. In the test 
set, there are 1,796 positively labeled links and 8,894 negatively labeled 
links.</P>
<P class="Para" id="Par21" xmlns="http://www.w3.org/1999/xhtml">The network 
evolves as new links are identified over time. In order to further examine the 
ability of the proposed framework to identify the evolution of network 
connectivity, a much larger set of DTIs were collected from the PubChem BioAssay 
database. PubChem BioAssay categorizes depositor-provided bioactivities between 
compounds and protein targets into active, inactive, and unspecified groups, 
according to assay descriptions and activity values. If the interactions between 
compounds and protein targets are categorized as active in PubChem BioAssay, and 
the active interaction pairs have reported activity values of less than 
1&nbsp;μM, the links are positively labeled; if the interactions between 
compounds and proteins are categorized as inactive in PubChem BioAssay, and 
there are reported activities for the interactions, the links are negatively 
labeled. A set of 145,622 positively labeled links contained in the current 
Chem2Bio2RDF semantic network, plus 600,000 negatively labeled links, constitute 
a training set; another set of 43,159 positively labeled links that are not 
contained in the current Chem2Bio2RDF semantic network, but are true positive 
DTIs, identified through bioassay experiments, plus195,000 negatively labeled 
links, comprise the test set. Since the positive DTIs in the test set were 
obtained after construction of the network, this independent test set is used to 
examine the ability to predict the links in the future network based on the 
topological features of the current network.</P></SECTION><SECTION class="Section2 RenderAsSection2" 
id="Sec6">
<H3 class="Heading">Binary classification models</H3>
<P class="Para" id="Par22" xmlns="http://www.w3.org/1999/xhtml">In order to 
demonstrate how well the similarity neighboring links obtained from PubChem 
databases can improve link prediction performance, we have constructed different 
machine learning models, based on two sets of path count topological features. 
Feature set <SPAN class="EmphasisTypeBoldItalic" xmlns="">I</SPAN> does not 
include any meta-paths involving similarity neighboring links, so it only 
contains 29 path count topological features. Feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">II</SPAN> includes all of the path counts encoded in 51 commuting 
matrixes. We also examined the improvement of predictive performance using an 
enriched topological feature space. RW normalization was applied to 51 path 
count topological features, and by combining the path counts and random walks, 
we obtained feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">III</SPAN>, which contains 102 topological features.</P>
<P class="Para" id="Par23" xmlns="http://www.w3.org/1999/xhtml">Two popular 
machine learning algorithms were investigated. Random forest (RF) represents a 
collection of decision trees, which are grown from bootstrap samples of the 
training data without pruning, and make predictions based on majority votes of 
the ensemble trees [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR29">29</A></SPAN>]. 
RF takes advantage of Out-of-Bag (OOB) error as an unbiased estimate of 
generalized test error, so there is no need to run cross-validation. RF can 
calculate the importance of features as well. The values for a given feature are 
permuted across all of the compound-protein pairs. Either classification 
accuracies or node impurities (Gini indexes) are measured before and after 
permutations, and the difference in the measures is used to evaluate feature 
importance. A default value for the number of trees was used (<EM class="EmphasisTypeItalic" 
xmlns="">ntree</EM> = 500) in the present study, which has been proven to be 
satisfactory in most cases [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR30">30</A></SPAN>]. 
The optimal value for tuning parameter <EM class="EmphasisTypeItalic" 
xmlns="">mtry</EM> was identified by a grid search.</P>
<P class="Para" id="Par24" xmlns="http://www.w3.org/1999/xhtml">In contrast to 
the tree-based model, Support Vector Machine (SVM) is based on a statistical 
learning theory derived from the structural risk minimization principle and 
Vapnik-Chervonenkis (VC) dimension [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR31">31</A></SPAN>]. 
A soft margin SVM with radial basis function (RBF) kernel in the Gaussian form 
was used in the present study. The optimal values for tuning parameters (<EM 
class="EmphasisTypeItalic" xmlns="">C</EM> and <EM class="EmphasisTypeItalic" 
xmlns="">λ</EM>) were determined by a grid search using 10-fold 
cross-validation.</P>
<DIV class="Para" id="Par25" xmlns="http://www.w3.org/1999/xhtml">The 
classification performances were evaluated using the F<SUB>1</SUB>-score [<SPAN 
class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR32">32</A></SPAN>], 
which is the harmonic mean of precision and recall.
<DIV class="Equation EquationMathjax" id="Equ1">
<DIV class="EquationContent">$$ 
{\mathrm{F}}_1\;\mathrm{score}:\;\frac{2TP}{2TP+FP+FN} $$</DIV>
<DIV class="EquationNumber">(1)</DIV></DIV></DIV>
<P class="Para" id="Par26" 
xmlns="http://www.w3.org/1999/xhtml">F<SUB>1</SUB>-score can be used for 
statistical hypothesis testing, in particular, for imbalanced datasets. Both RF 
and SVM can calculate the probabilities of classifications, and rankings can be 
derived from the probability calculations. The predictive performance on 
rankings was evaluated according to Receiver Operating Characteristic (ROC) and 
Precision Recall (PR) curves for all of the models. The area under the curve for 
ROC (AUCROC) and PR (AUCPR) were calculated using the natural spline 
interpolation encoded in the R package ‘Miscellaneous Esoteric Statistical 
Scripts’ (MESS). The early hit recognitions that are considered more important 
in virtual screening experiments were evaluated using Boltzmann-enhanced 
discrimination of ROC (BEDROC), which was calculated using the R package 
‘enrichvs.’</P></SECTION></DIV></SECTION><SECTION class="Section1 RenderAsSection1 SectionTypeDiscussion" 
id="Sec7" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Results and discussion</H2>
<DIV class="js-CollapseSection">
<DIV class="Para" id="Par27" xmlns="http://www.w3.org/1999/xhtml">The optimal 
tuning parameters and the statistical results for all the binary classification 
models are summarized in Table&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Tab3">3</A></SPAN>. 
RF outperformed SVM across all three feature sets. Both RF and SVM yielded 
consistent rankings of the predictive performance for the different feature 
sets: feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">III</SPAN> &gt; feature set <SPAN class="EmphasisTypeBoldItalic" xmlns="">II</SPAN> &gt; feature 
set <SPAN class="EmphasisTypeBoldItalic" xmlns="">I</SPAN>. The similarity 
neighboring links improved the link prediction performance on test set by 
5.5&nbsp;% in RF models, and by around 4.4&nbsp;% in SVM models. In combination 
with RW normalization, the predictive performance of RF models was improved by 
2&nbsp;%, and the predictive performance of SVM models were boosted by 
3.5&nbsp;%. The differences in predictive performance were consistently 
demonstrated by ROC and PR curves as well (see Fig.&nbsp;<SPAN class="InternalRef"><A 
href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Fig2">2</A></SPAN>). 
The ROC space and PR space agreed on the rankings of different feature sets, in 
terms of predictive performance. We can see that feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">III</SPAN> dominated both ROC space and PR space for both RF and SVM 
models, and RF models slightly outperformed SVM models. Since we have imbalanced 
distributions for positive and negative labels, PR curves can provide better 
visual representations than ROC curves to identify the difference of predictive 
performance. As shown in Fig.&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Fig2">2</A></SPAN>, 
the ROC curves were closely clustered, and the PR curves for different models 
were separated to a larger extent. The differences among AUCPRs were larger than 
the differences among AUCROCs, as well (see Table&nbsp;<SPAN 
class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Tab4">4</A></SPAN>). 
It is clear that similarity neighboring links are important for link prediction 
in the semantic network, and RW normalization can boost predictive performance 
by enriching feature space. It is noteworthy that all the machine learning 
models performed fairly well on both training and test sets without 
over-fitting. In addition, both feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">II</SPAN> and feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">III</SPAN> produced AUCROCs greater than 0.92, which was produced by 
SLAP [<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR15">15</A></SPAN>]. 
Hence, meta-path-based topological features have been proven to be valuable for 
link prediction in complex semantic networks using machine learning 
models.<FIGURE class="FigureTable" id="Tab3" xmlns=""><FIGCAPTION class="Caption" 
lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Table 
3</SPAN>                       
<P class="SimplePara" xmlns="">Statistics of binary classification 
models&nbsp;built upon different feature sets and using different machine 
learning algorithms</P></DIV></FIGCAPTION>
<DIV class="Table">
<TABLE border="1" xmlns="http://www.w3.org/1999/xhtml">
  <COLGROUP>
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="left">
  <COL align="char" char="."></COLGROUP>
  <THEAD>
  <TR class="header">
    <TH align="left" rowspan="2">
      <P class="SimplePara" xmlns="">topological feature</P></TH>
    <TH align="left" rowspan="2">
      <P class="SimplePara" xmlns="">Dataset</P></TH>
    <TH align="left" colspan="2">
      <P class="SimplePara" xmlns="">Random Forest</P></TH>
    <TH align="left" colspan="3">
      <P class="SimplePara" xmlns="">Support Vector Machine</P></TH></TR>
  <TR class="header">
    <TH align="left">
      <P class="SimplePara" xmlns=""><EM class="EmphasisTypeItalic">mtry</EM>    
                             </P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns="">F<SUB 
      xmlns="http://www.w3.org/1999/xhtml">1</SUB>-score</P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns=""><EM class="EmphasisTypeItalic">C</EM>       
                          </P></TH>
    <TH align="left">
      <P class="SimplePara" xmlns=""><EM class="EmphasisTypeItalic">λ</EM>       
                          </P></TH>
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">F<SUB 
      xmlns="http://www.w3.org/1999/xhtml">1</SUB>-score</P></TH></TR></THEAD>
  <TBODY>
  <TR class="noclass">
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">I</SPAN> 
                                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">Training</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">12</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">0.780</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">8</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">0.250</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.766</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Test</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">0.735</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.719</P></TD></TR>
  <TR class="noclass">
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">II</SPAN> 
                                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">Training</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">13</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">0.844</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">16</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">0.062</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.810</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Test</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">0.790</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.763</P></TD></TR>
  <TR class="noclass">
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">III</SPAN> 
                                             </P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">Training</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">13</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">0.859</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">16</P></TD>
    <TD align="left" rowspan="2">
      <P class="SimplePara" xmlns="">0.016</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.843</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Test</P></TD>
    <TD align="left">
      <P class="SimplePara" xmlns="">0.810</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" 
xmlns="">0.798</P></TD></TR></TBODY></TABLE></DIV></FIGURE><FIGURE class="Figure" 
id="Fig2" xmlns="">
<DIV class="MediaObject" id="MO2" xmlns="http://www.w3.org/1999/xhtml"><IMG alt="https://static-content.springer.com/image/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_Fig2_HTML.gif" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/12859_2016_1005_Fig2_HTML.gif"> 
                  </DIV><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Fig. 
2</SPAN>                       
<P class="SimplePara" xmlns="">Receiver operating characteristic curves (<STRONG 
class="EmphasisTypeBold">a</STRONG>) and precision/recall curves (<STRONG class="EmphasisTypeBold">b</STRONG>) 
for the six models using two machine learning algorithms to build binary 
classification models upon three topological feature spaces. RF means Random 
Forest, SVM means support vector machine, FI means feature set <SPAN class="EmphasisTypeBoldItalic">I</SPAN>, 
FII means feature set <SPAN class="EmphasisTypeBoldItalic">II</SPAN>, and FIII 
means feature set <SPAN class="EmphasisTypeBoldItalic">III</SPAN>                
                  </P></DIV></FIGCAPTION></FIGURE><FIGURE class="FigureTable" 
id="Tab4" xmlns=""><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Table 
4</SPAN>                       
<P class="SimplePara" xmlns="">Area under ROC curve (AUCROC) and area under PR 
curve (AUCPR) of random forest and support vector machine classification models 
using different feature sets</P></DIV></FIGCAPTION>
<DIV class="Table">
<TABLE border="1" xmlns="http://www.w3.org/1999/xhtml">
  <COLGROUP>
  <COL align="left">
  <COL align="char" char=".">
  <COL align="char" char=".">
  <COL align="char" char=".">
  <COL align="char" char="."></COLGROUP>
  <THEAD>
  <TR class="header">
    <TH align="left" rowspan="2">
      <P class="SimplePara" xmlns="">topological feature</P></TH>
    <TH align="char" colspan="2" char=".">
      <P class="SimplePara" xmlns="">Random Forest</P></TH>
    <TH align="char" colspan="2" char=".">
      <P class="SimplePara" xmlns="">Support Vector Machine</P></TH></TR>
  <TR class="header">
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">AUCROC</P></TH>
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">AUCPR</P></TH>
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">AUCROC</P></TH>
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">AUCPR</P></TH></TR></THEAD>
  <TBODY>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">I</SPAN> 
                                             </P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.891</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.772</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.871</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.729</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">II</SPAN> 
                                             </P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.927</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.826</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.905</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.768</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">III</SPAN> 
                                             </P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.938</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.857</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.922</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" 
xmlns="">0.795</P></TD></TR></TBODY></TABLE></DIV></FIGURE></DIV>
<DIV class="Para" id="Par28" xmlns="http://www.w3.org/1999/xhtml">In order to 
further compare the proposed approached with SLAP, we carried out link 
predictions using both methods on a large set of unknown links of an evolving 
semantic network. The labels of those unknown links were derived from 
experimental evidence deposited in PubChem BioAssay databases after the 
Chem2Bio2RDF network was constructed. Hence, these positive labels can be viewed 
as experimental validations when assessing link prediction performance. The 
proposed framework, using RF to build a binary classification model upon feature 
set <SPAN class="EmphasisTypeBoldItalic" xmlns="">III,</SPAN> yielded much 
better BEDROC and AUCROC than SLAP (Table&nbsp;<SPAN class="InternalRef"><A 
href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Tab5">5</A></SPAN>). 
BEDROC is mainly used to compare ranking systems in terms of early recognition 
[<SPAN class="CitationRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#CR33">33</A></SPAN>]. 
Our approach yielded much better AUC of BEDROC using a default coefficient 
parameter (α = 20.0) (Table&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Tab5">5</A></SPAN>). 
The difference can be seen in Fig.&nbsp;<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Fig3">3</A></SPAN> 
as well.<FIGURE class="FigureTable" id="Tab5" xmlns=""><FIGCAPTION class="Caption" 
lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Table 
5</SPAN>                       
<P class="SimplePara" xmlns="">Comparing the proposed framework (random forest 
classification model applied on feature set III) with existing algorithm SLAP 
using Area under ROC curve (AUCROC) and area under PR curve 
(AUCPR)</P></DIV></FIGCAPTION>
<DIV class="Table">
<TABLE border="1" xmlns="http://www.w3.org/1999/xhtml">
  <COLGROUP>
  <COL align="left">
  <COL align="char" char=".">
  <COL align="char" char="."></COLGROUP>
  <THEAD>
  <TR class="header">
    <TH align="left">&nbsp;</TH>
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">AUCROC</P></TH>
    <TH align="char" char=".">
      <P class="SimplePara" xmlns="">BEDROC</P></TH></TR></THEAD>
  <TBODY>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">Feature set <SPAN class="EmphasisTypeBoldItalic">III</SPAN> 
                                             </P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.845</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.929</P></TD></TR>
  <TR class="noclass">
    <TD align="left">
      <P class="SimplePara" xmlns="">SLAP</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" xmlns="">0.670</P></TD>
    <TD align="char" char=".">
      <P class="SimplePara" 
xmlns="">0.672</P></TD></TR></TBODY></TABLE></DIV></FIGURE><FIGURE class="Figure" 
id="Fig3" xmlns="">
<DIV class="MediaObject" id="MO3" xmlns="http://www.w3.org/1999/xhtml"><IMG alt="https://static-content.springer.com/image/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_Fig3_HTML.gif" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/12859_2016_1005_Fig3_HTML.gif"> 
                  </DIV><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Fig. 
3</SPAN>                       
<P class="SimplePara" xmlns="">ROC curves for the Random Forest model built upon 
feature set <SPAN class="EmphasisTypeBoldItalic">III</SPAN> and SLAP. RF means 
Random Forest and FIII means feature set <SPAN 
class="EmphasisTypeBoldItalic">III</SPAN>                                  
</P></DIV></FIGCAPTION></FIGURE></DIV>
<DIV class="Para" id="Par29" xmlns="http://www.w3.org/1999/xhtml">By applying 
the intrinsic feature ranking algorithm of the RF on feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">II</SPAN>, we can tell which meta-paths are important for link 
prediction. Feature importance can be visualized as a dot plot (Fig.&nbsp;<SPAN 
class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Fig4">4</A></SPAN>). 
Two measures evaluated before and after permutations were used for feature 
ranking: decrease of classification accuracy and decrease of Gini index. 
Although two measures do not always agree on which features are important, we 
still can identify some significantly important meta-paths according to two 
measures. The top four important meta-paths were C1, C19, C16, and C39, and the 
network nodes connected by these important meta-paths are compounds, proteins, 
and GO annotations. It is noteworthy that the top three important meta-paths 
only contain semantic links between compounds and proteins, and the top two 
important meta-paths contain similarity neighboring links. Therefore, semantic 
links between compounds and proteins, including similarity neighboring links and 
interaction links, played a major role in predicting CPIs.<FIGURE class="Figure" 
id="Fig4" xmlns="">
<DIV class="MediaObject" id="MO4" xmlns="http://www.w3.org/1999/xhtml"><IMG alt="https://static-content.springer.com/image/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_Fig4_HTML.gif" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/12859_2016_1005_Fig4_HTML.gif"> 
                  </DIV><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Fig. 
4</SPAN>                       
<P class="SimplePara" xmlns="">Variable importance for Random Forest model built 
with feature set <SPAN class="EmphasisTypeBoldItalic">II</SPAN>. The color code 
for feature importance according to mean decrease accuracy: red (&gt;70), blue 
(&gt;45 and &lt;70), green (&lt;45); the color code for feature importance 
according to mean decrease Gini index: red (&gt;240), blue (&gt;240 and 
&lt;100), green (&lt;100)</P></DIV></FIGCAPTION></FIGURE></DIV>
<DIV class="Para" id="Par30" xmlns="http://www.w3.org/1999/xhtml">In contrast to 
SLAP, that pre-calculates feature importance before making predictions, the 
proposed framework can evaluate feature importance and build predictive models 
at the same time. The importance of a given topological feature may vary to some 
extent when different sets of training data are considered, or when new links 
are added into the network as a function of time. We carried out an experiment 
to demonstrate that feature importance may vary significantly when different 
sets of data are used to build predictive models. We constructed 1,000 RF models 
using randomly selected training sets with feature set <SPAN class="EmphasisTypeBoldItalic" 
xmlns="">II</SPAN>. Each training set was compiled by 100 positively labeled 
links from the DrugBank set, and 100 negatively labeled links from the PubChem 
BioAssay set with experimental bioactivity value greater than 10&nbsp;μM. The 
changes of feature importance in different models can be seen in Fig.&nbsp;<SPAN 
class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Fig5">5</A></SPAN>. 
It is clearly that feature importance varied a lot in different models. Feature 
C4 has the smallest standard deviation (0.828) and feature C39 has the largest 
standard deviation (5.537). It is noteworthy that all of the top four importance 
features in the aforementioned models (C1, C16, C19, and C39) have very large 
standard deviations. Even though their importance varied a lot in different 
models, their mean values were well above the average of others; in particular, 
the mean values of C1 and C39 were much larger than those of other topological 
features. The predictive performances of those 1,000 RF models tested against a 
randomly selected set of 50 positive labels and 50 negative labels (not included 
in any of those 1,000 training sets) varied a lot as well. The highest 
F<SUB>1</SUB>-score is 0.937 and the lowest F<SUB>1</SUB>-score is 0.667. Hence, 
the selection of training set is also very important to build highly predictive 
machine learning models.<FIGURE class="Figure" id="Fig5" xmlns="">
<DIV class="MediaObject" id="MO5" xmlns="http://www.w3.org/1999/xhtml"><IMG alt="https://static-content.springer.com/image/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_Fig5_HTML.gif" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/12859_2016_1005_Fig5_HTML.gif"> 
                  </DIV><FIGCAPTION class="Caption" lang="en">
<DIV class="CaptionContent" xmlns="http://www.w3.org/1999/xhtml"><SPAN class="CaptionNumber">Fig. 
5</SPAN>                       
<P class="SimplePara" xmlns="">Box plot for the variable importance varying in 1 
000 Random Forest 
models</P></DIV></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION class="Section1 RenderAsSection1" 
id="Sec8" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Conclusions</H2>
<DIV class="js-CollapseSection">
<P class="Para" id="Par31" xmlns="http://www.w3.org/1999/xhtml">The semantic 
network integrating domain knowledge across chemical and biological space can be 
leveraged for large-scale data mining. Among the different kinds of semantic 
links, drug-target connectivity maps have drawn extensive attention, since they 
are beneficial for drug discovery and development, in particular, drug 
repositioning and polypharmacology research. In the present work, we have 
proposed a framework to construct state-of-the-art machine learning models using 
meta-path-based topological features for link prediction in complex semantic 
networks. Supervised classification models were shown to be powerful, based on 
their predictive performance in an independent test set containing links of an 
evolving network. In addition, the intrinsic feature ranking algorithm embedded 
in machine learning models can be used to select the most important topological 
features. Although the proposed framework was only applied to predict DTIs in 
the present work, it can definitely be used for other purposes, such as to 
predict associations between drugs and adverse side effects, as well as 
associations between proteins and diseases. In the future, we want to study how 
to select the most relevant training set for a given prediction task, and how 
much training set selection can improve predictive performance.</P><SECTION 
class="Section2 RenderAsSection2" id="Sec9">
<H3 class="Heading">Availability of Data and Materials</H3>
<P class="Para" id="Par111" xmlns="http://www.w3.org/1999/xhtml">The data sets 
supporting the results of this article are included within the article and its 
additional files (Additional files <SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#MOESM1">1</A></SPAN>, 
<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#MOESM2">2</A></SPAN>, 
<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#MOESM3">3</A></SPAN>, 
<SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#MOESM4">4</A></SPAN> 
and <SPAN class="InternalRef"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#MOESM5">5</A></SPAN>).</P></SECTION></DIV></SECTION><SECTION 
class="AbbreviationGroup" xmlns:meta="http://www.springer.com/app/meta" 
xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Abbreviations</H2>
<DIV class="js-CollapseSection">
<DIV class="DefinitionList">
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>FN:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par4" xmlns="http://www.w3.org/1999/xhtml">false 
  negative</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>FP:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par5" xmlns="http://www.w3.org/1999/xhtml">false 
  positive</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>MCC:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par6" xmlns="http://www.w3.org/1999/xhtml">Mathews 
  correlation coefficient</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>PC:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par7" xmlns="http://www.w3.org/1999/xhtml">path 
  count</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>RF:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par8" xmlns="http://www.w3.org/1999/xhtml">random 
  forest</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>RW:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par9" xmlns="http://www.w3.org/1999/xhtml">random 
  walk</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>SLAP:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par10" xmlns="http://www.w3.org/1999/xhtml">semantic link 
  association prediction</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>TN:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par11" xmlns="http://www.w3.org/1999/xhtml">true 
  negative</P></DD></DL>
<DL class="DefinitionListEntry">
  <DT class="Term"><DFN>TP:&nbsp;</DFN></DT>
  <DD class="Description">
  <P class="Para" id="Par12" xmlns="http://www.w3.org/1999/xhtml">true 
  positive</P></DD></DL></DIV></DIV></SECTION><SECTION class="Section1 RenderAsSection1" 
id="Declarations" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">              Declarations          
   </H2>
<DIV class="js-CollapseSection">
<DIV class="Acknowledgments" xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Acknowledgements</H3>
<P class="SimplePara" xmlns="">This research was supported [in part] by the 
Intramural Research Program of the National Library of Medicine, NIH. Many 
thanks to the PubChem team, who provided database API functions to retrieve data 
from Compound and BioAssay databases. Many thanks to Ming Hao in the discussion 
about R packages and calculations.&nbsp;Many thanks to Jeremy Yang in the 
discussion and help to maintain Chem2Bio2RDF web server.</P></DIV>
<DIV class="License" xmlns="http://www.w3.org/1999/xhtml">
<P class="SimplePara" xmlns=""><STRONG class="EmphasisTypeBold">Open 
Access</STRONG>This article is distributed under the terms of the Creative 
Commons Attribution 4.0 International License (<SPAN class="ExternalRef" xmlns="http://www.w3.org/1999/xhtml"><A 
href="http://creativecommons.org/licenses/by/4.0/"><SPAN 
class="RefSource">http://​creativecommons.​org/​licenses/​by/​4.​0/​</SPAN></A></SPAN>), 
which permits unrestricted use, distribution, and reproduction in any medium, 
provided you give appropriate credit to the original author(s) and the source, 
provide a link to the Creative Commons license, and indicate if changes were 
made. The Creative Commons Public Domain Dedication waiver (<SPAN class="ExternalRef" 
xmlns="http://www.w3.org/1999/xhtml"><A href="http://creativecommons.org/publicdomain/zero/1.0/"><SPAN 
class="RefSource">http://​creativecommons.​org/​publicdomain/​zero/​1.​0/​</SPAN></A></SPAN>) 
applies to the data made available in this article, unless otherwise 
stated.</P></DIV><ASIDE class="Appendix" id="App1">
<H3 class="Heading">Additional files</H3>
<DIV class="Para" id="Par55" xmlns="http://www.w3.org/1999/xhtml">
<DIV class="MediaObject" id="MOESM1">
<DIV class="esm-item " xmlns="">
<DIV class="caption-container" xmlns="http://www.w3.org/1999/xhtml"><A href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_MOESM1_ESM.zip"><SPAN 
class="caption-number">Additional file 1:</SPAN></A>                           
<SPAN class="SimplePara" xmlns="">(Cpluspluscodes.zip): it contains the C++ 
codes to generate adjacent matrix and compute commuting matrix. (ZIP 9294 
kb)</SPAN>                         </DIV></DIV></DIV>
<DIV class="MediaObject" id="MOESM2">
<DIV class="esm-item " xmlns="">
<DIV class="caption-container" xmlns="http://www.w3.org/1999/xhtml"><A href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_MOESM2_ESM.zip"><SPAN 
class="caption-number">Additional file 2:</SPAN></A>                           
<SPAN class="SimplePara" xmlns="">(Rcodes.zip): it contains the R codes to apply 
machine learning algorithms on the commuting matrix feature space to construct 
binary classification model. (ZIP 48844 kb)</SPAN>                         
</DIV></DIV></DIV>
<DIV class="MediaObject" id="MOESM3">
<DIV class="esm-item " xmlns="">
<DIV class="caption-container" xmlns="http://www.w3.org/1999/xhtml"><A href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_MOESM3_ESM.zip"><SPAN 
class="caption-number">Additional file 3:</SPAN></A>                           
<SPAN class="SimplePara" xmlns="">(semantic_network_dataset.zip): it contains 
the semantic links of the augmented Chem2Bio2RDF graph and the labels for 
internal test set, which has been blinded out from training set. (ZIP 26931 
kb)</SPAN>                         </DIV></DIV></DIV>
<DIV class="MediaObject" id="MOESM4">
<DIV class="esm-item " xmlns="">
<DIV class="caption-container" xmlns="http://www.w3.org/1999/xhtml"><A href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_MOESM4_ESM.zip"><SPAN 
class="caption-number">Additional file 4:</SPAN></A>                           
<SPAN class="SimplePara" xmlns="">(metapath_commuting_matrix.zip): it contains 
the calculated commuting matrix for internal and external validating. The 
internal test set was used by SLAP in the original publication; the external 
test set was prepared in the present study and intended to examine the ability 
to predict evolving network topologies. (ZIP 27340 kb)</SPAN>                    
     </DIV></DIV></DIV>
<DIV class="MediaObject" id="MOESM5">
<DIV class="esm-item " xmlns="">
<DIV class="caption-container" xmlns="http://www.w3.org/1999/xhtml"><A href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12859-016-1005-x/MediaObjects/12859_2016_1005_MOESM5_ESM.zip"><SPAN 
class="caption-number">Additional file 5:</SPAN></A>                           
<SPAN class="SimplePara" xmlns="">(evolving_network_prediction.zip): it contains 
the predicted results using SLAP and presented meta-path-based semantic network 
analysis framework. (ZIP 4712 kb)</SPAN>                         
</DIV></DIV></DIV></DIV></ASIDE><ASIDE class="ArticleNote ArticleNoteMisc" 
xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Competing interests</H3>
<P class="SimplePara" xmlns="">The authors declare that they have no competing 
interests.</P></ASIDE><ASIDE class="ArticleNote ArticleNoteMisc" xmlns="http://www.w3.org/1999/xhtml">
<H3 class="Heading" xmlns="">Authors’ contributions</H3>
<P class="SimplePara" xmlns="">GF designed and implemented the framework; YD 
initialized the research project; CB, YS, and EB contributed to the data 
analysis, interpretation, and validation. All authors have contributed to 
manuscript editing. All authors read and approved the final 
manuscript.</P></ASIDE></DIV></SECTION></DIV>
<DIV class="FulltextWrapper"><SECTION class="Section1 RenderAsSection1 Affiliations" 
id="Aff" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Authors’ Affiliations</H2>
<DIV class="js-CollapseSection">
<DIV class="Affiliation" id="Aff1"><SPAN class="AffiliationNumber">(1)</SPAN>
<DIV class="AffiliationText">National Center for Biotechnology Information, 
National Library of Medicine, National Institutes of Health</DIV></DIV>
<DIV class="Affiliation" id="Aff2"><SPAN class="AffiliationNumber">(2)</SPAN>
<DIV class="AffiliationText">School of Informatics &amp; Computing, Indiana 
University</DIV></DIV>
<DIV class="Affiliation" id="Aff3"><SPAN class="AffiliationNumber">(3)</SPAN>
<DIV class="AffiliationText">School of Information Management, Wuhan University, 
Wuchang</DIV></DIV>
<DIV class="Affiliation" id="Aff4"><SPAN class="AffiliationNumber">(4)</SPAN>
<DIV class="AffiliationText">Department of Medicine, Stanford 
University</DIV></DIV>
<DIV class="Affiliation" id="Aff5"><SPAN class="AffiliationNumber">(5)</SPAN>
<DIV class="AffiliationText">College of Computer and Information Science, 
Northeastern University</DIV></DIV></DIV></SECTION></DIV>
<DIV class="FulltextWrapper"><SECTION class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods" 
id="Bib1" xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<H2 class="Heading js-ToggleCollapseSection">References</H2>
<DIV class="js-CollapseSection">
<OL class="BibliographyWrapper">
  <LI class="Citation"><CITE class="CitationContent" id="CR1">Bredel M, Jacoby 
  E. Chemogenomics: an emerging strategy for rapid target and drug discovery. 
  Nat Rev Genet. 2004;5(4):262–75.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1038/nrg1317"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15131650"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR2">Yamanishi Y. 
  Chemogenomic approaches to infer drug-target interaction networks. Methods Mol 
  Biol. 2013;939:97–113.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1007/978-1-62703-107-3_9"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23192544"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR3">Oprea TI, May EE, 
  Leitao A, Tropsha A. Computational systems chemical biology. Methods Mol Biol. 
  2011;672:459–88.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1007/978-1-60761-839-3_18"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20838980"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3547368"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR4">Oprea TI, Tropsha 
  A, Faulon JL, Rintoul MD. Systems chemical biology. Nat Chem Biol. 
  2007;3(8):447–50.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1038/nchembio0807-447"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17637771"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2734506"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR5">Lee S, Park K, Kim 
  D. Building a drug-target network and its applications. Expert Opin Drug 
  Discov. 2009;4(11):1177–89.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1517/17460440903322234"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23480435"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR6">Kotelnikova E, 
  Yuryev A, Mazo I, Daraselia N. Computational approaches for drug repositioning 
  and combination therapy design. J Bioinform Comput Biol. 
  2010;8(3):593–606.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1142/S0219720010004732"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20556864"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR7">Li J, Zhu X, Chen 
  JY. Building disease-specific drug-protein connectivity maps from molecular 
  interaction networks and PubMed abstracts. PLoS Comput Biol. 
  2009;5(7):e1000450.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1371/journal.pcbi.1000450"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19649302"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2709445"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR8">Xie L, Li J, 
  Bourne PE. Drug discovery using chemical systems biology: identification of 
  the protein-ligand binding network to explain the side effects of CETP 
  inhibitors. PLoS Comput Biol. 2009;5(5):e1000387.<SPAN 
  class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A href="http://dx.doi.org/10.1371/journal.pcbi.1000387"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19436720"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2676506"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR9">Wild DJ, Ding Y, 
  Sheth AP, Harland L, Gifford EM, Lajiness MS. Systems chemical biology and the 
  Semantic Web: what they mean for the future of drug discovery research. Drug 
  Discov Today. 2012;17(9–10):469–74.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1016/j.drudis.2011.12.019"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22222943"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR10">Fu G, Batchelor 
  C, Dumontier M, Hastings J, Willighagen E, Bolton E. PubChemRDF: towards the 
  semantic annotation of PubChem compound and substance databases. J Cheminform. 
  2015;7:34.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1186/s13321-015-0084-4"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26175801"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4500850"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR11">Chen B, Dong X, 
  Jiao D, Wang H, Zhu Q, Ding Y, Wild DJ. Chem2Bio2RDF: a semantic framework for 
  linking and data mining chemogenomic and systems chemical biology data. BMC 
  Bioinformatics. 2010;11:255.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1186/1471-2105-11-255"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20478034"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881087"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR12">Callahan A, 
  Cruz-Toledo J, Ansell P, Dumontier M. Bio2RDF Release 2: Improved Coverage, 
  Interoperability and Provenance of Life Science Linked Data. In: Cimiano P, 
  Corcho O, Presutti V, Hollink L, Rudolph S, editors. The Semantic Web: 
  Semantics and Big Data vol. 7882. Berlin Heidelberg: Springer; 2013. p. 
  200–12.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1007/978-3-642-38288-8_14"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR13">Williams AJ, 
  Harland L, Groth P, Pettifer S, Chichester C, Willighagen EL, Evelo CT, 
  Blomberg N, Ecker G, Goble C, et al. Open PHACTS: semantic interoperability 
  for drug discovery. Drug Discov Today. 2012;17(21–22):1188–98.<SPAN class="Occurrences"><SPAN 
  class="Occurrence OccurrenceDOI"><A href="http://dx.doi.org/10.1016/j.drudis.2012.05.016"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22683805"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR14">Willighagen EL, 
  Waagmeester A, Spjuth O, Ansell P, Williams AJ, Tkachenko V, Hastings J, Chen 
  B, Wild DJ. The ChEMBL database as linked open data. J Cheminform. 
  2013;5(1):23.<SPAN class="Occurrences"><SPAN 
  class="Occurrence OccurrenceDOI"><A href="http://dx.doi.org/10.1186/1758-2946-5-23"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23657106"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3700754"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR15">Chen B, Ding Y, 
  Wild DJ. Assessing drug target association using semantic linked data. PLoS 
  Comput Biol. 2012;8(7):e1002574.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1371/journal.pcbi.1002574"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22859915"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3390390"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR16">Getoor L, Diehl 
  CP. Link mining: a survey. SIGKDD Explor Newsl. 2005;7(2):3–12.<SPAN class="Occurrences"><SPAN 
  class="Occurrence OccurrenceDOI"><A href="http://dx.doi.org/10.1145/1117454.1117456"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR17">Liben-Nowell D, 
  Kleinberg J. The link-prediction problem for social networks. J Am Soc Inf Sci 
  Technol. 2007;58(7):1019–31.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1002/asi.20591"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR18">Yamanishi Y, 
  Kotera M, Kanehisa M, Goto S. Drug-target interaction prediction from 
  chemical, genomic and pharmacological data in an integrated framework. 
  Bioinformatics. 2010;26(12):i246–54.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1093/bioinformatics/btq176"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20529913"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881361"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR19">Cheng F, Liu C, 
  Jiang J, Lu W, Li W, Liu G, Zhou W, Huang J, Tang Y. Prediction of drug-target 
  interactions and drug repositioning via network-based inference. PLoS Comput 
  Biol. 2012;8(5):e1002503.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1371/journal.pcbi.1002503"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22589709"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3349722"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR20">Cheng F, Zhou Y, 
  Li W, Liu G, Tang Y. Prediction of chemical-protein interactions network with 
  weighted network-based inference method. PLoS One. 2012;7(7):e41064.<SPAN 
  class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A href="http://dx.doi.org/10.1371/journal.pone.0041064"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22815915"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3397956"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR21">Cheng F, Li W, Wu 
  Z, Wang X, Zhang C, Li J, Liu G, Tang Y. Prediction of polypharmacological 
  profiles of drugs by the integration of chemical, side effect, and therapeutic 
  space. J Chem Inf Model. 2013;53(4):753–62.<SPAN class="Occurrences"><SPAN 
  class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1021/ci400010x"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23527559"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR22">Sun Y, Han J, 
  Aggarwal CC, Chawla NV: When will it happen?: relationship prediction in 
  heterogeneous information networks. In: Proceedings of the fifth ACM 
  international conference on Web search and data mining; Seattle, Washington, 
  USA. 2124373: ACM 2012: 663–672.</CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR23">Sun Y, Han J, Yan 
  X, Yu PS, Wu T. PathSim: Meta Path-Based Top-K Similarity Search in 
  Heterogeneous Information Networks. PVLDB. 2011;4(11):992–1003.</CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR24">PubChem 2D 
  similarity neighbor. <SPAN class="ExternalRef" xmlns="http://www.w3.org/1999/xhtml"><A 
  href="https://pubchem.ncbi.nlm.nih.gov/score_matrix/score_matrix-help.html"><SPAN 
  class="RefSource">https://​pubchem.​ncbi.​nlm.​nih.​gov/​score_​matrix/​score_​matrix-help.​html</SPAN></A></SPAN> 
  [<SPAN class="ExternalRef" xmlns="http://www.w3.org/1999/xhtml"><A href="https://pubchem.ncbi.nlm.nih.gov/score_matrix/score_matrix-help.html"><SPAN 
  class="RefSource">https://​pubchem.​ncbi.​nlm.​nih.​gov/​score_​matrix/​score_​matrix-help.​html</SPAN></A></SPAN>]. 
  Accessed 4 Apr 2016. </CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR25">Kim S, Thiessen 
  PA, Bolton EE, Chen J, Fu G, Gindulyte A, Han L, He J, He S, Shoemaker BA, 
  Wang J, Yu B, Zhang J, Bryant SH. PubChem Substance and Compound databases. 
  Nucleic Acids Res. 2016;44(D1):D1202-13. doi:<SPAN class="ExternalRef" xmlns="http://www.w3.org/1999/xhtml"><A 
  href="http://dx.doi.org/10.1186/s10194-015-0575-8"><SPAN 
  class="RefSource">10.​1093/​nar/​gkv951</SPAN></A></SPAN>.</CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR26">Wang Y, Xiao J, 
  Suzek TO, Zhang J, Wang J, Bryant SH. PubChem: a public information system for 
  analyzing bioactivities of small molecules. Nucleic Acids Res. 2009;37(Web 
  Server issue):W623–633.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1093/nar/gkp456"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19498078"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2703903"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR27">Conrad SA: An 
  Open Source C++ Linear Algebra Library for Fast Prototyping and 
  Computationally Intensive Experiments. Technical Report, NICTA, 2010. <SPAN 
  class="ExternalRef" xmlns="http://www.w3.org/1999/xhtml"><A href="http://arma.sourceforge.net/armadillo_nicta_2010.pdf"><SPAN 
  class="RefSource">http://​arma.​sourceforge.​net/​armadillo_​nicta_​2010.​pdf</SPAN></A></SPAN>.</CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR28">Wang Y, Xiao J, 
  Suzek TO, Zhang J, Wang J, Zhou Z, Han L, Karapetyan K, Dracheva S, Shoemaker 
  BA, et al. PubChem’s BioAssay Database. Nucleic Acids Res. 2012;40(Database 
  issue):D400–412.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1093/nar/gkr1132"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22140110"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN><SPAN 
  class="Occurrence OccurrencePMCID"><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245056"><SPAN><SPAN>PubMed 
  Central</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR29">Breiman L. Random 
  forests. Mach Learn. 2001;45:5–32.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1023/A%3A1010933404324"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR30">Fu G, Liu S, Nan 
  X, Dale OR, Zhao Z, Chen Y, Wilkins DE, Manly SP, Cutler SJ, Doerksen RJ. 
  Quantitative Structure-Activity Relationship Analysis and a Combined 
  Ligand-Based/Structure-Based Virtual Screening Study for Glycogen Synthase 
  Kinase-3. Mol Inform. 2014;33(9):627–40.<SPAN class="Occurrences"><SPAN class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1002/minf.201400045"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN></SPAN></CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR31">Cortes C, Vapnik 
  V. SUPPORT-VECTOR NETWORKS. Mach Learn. 1995;20(3):273–97.</CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR32">Powers DMW: 
  Evaluation: From Precision, Recall and F-Factor to ROC, Informedness, 
  Markedness &amp; Correlation. In. Adelaide: School of Informatics and 
  Engineering, Flinders University of South Australia; Australia: 
  2007.</CITE></LI>
  <LI class="Citation"><CITE class="CitationContent" id="CR33">Truchon JF, Bayly 
  CI. Evaluating virtual screening methods: good and bad metrics for the “early 
  recognition” problem. J Chem Inf Model. 2007;47(2):488–508.<SPAN class="Occurrences"><SPAN 
  class="Occurrence OccurrenceDOI"><A 
  href="http://dx.doi.org/10.1021/ci600426e"><SPAN><SPAN>View 
  Article</SPAN></SPAN></A></SPAN><SPAN class="Occurrence OccurrencePID"><A 
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17288412"><SPAN><SPAN>PubMed</SPAN></SPAN></A></SPAN></SPAN></CITE></LI></OL></DIV></SECTION><SECTION 
class="Section1 RenderAsSection1" id="CopyrightMessage" xmlns:meta="http://www.springer.com/app/meta" 
xmlns="">
<H2 class="Heading js-ToggleCollapseSection">Copyright</H2>
<DIV class="js-CollapseSection">
<DIV class="ArticleCopyright" xmlns="http://www.w3.org/1999/xhtml">©&nbsp;Fu et 
al.&nbsp;2016</DIV></DIV></SECTION></DIV></DIV>
<DIV class="Main_sidebar Main_sidebar--right"><ASIDE role="complementary">
<DIV class="SideBox">
<P class="u-marginBtmM"><A class="button button--size-300 button--priority-1 button--full" 
id="articlePdf" href="http://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1005-x?site=bmcbioinformatics.biomedcentral.com" 
data-event-label="10.1186/s12859-016-1005-x" data-event-action="Click download article PDF" 
data-event-category="PDF Download" data-track-event="click">Download PDF</A>     
</P>
<DIV class="Collapse Collapse--button" id="js-exportCitations">
<H3 class="Collapse_toggle Collapse_toggle--right" data-toggle="collapse" 
data-target="js-exportCitations">Export citations</H3>
<DIV class="Panel Panel--s Collapse_content">
<DIV class="Panel_section">
<H4 class="Panel_sectionTitle">Citations &amp; References</H4>
<UL class="ListDownload" id="articleReferences">
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=refman&amp;flavour=full" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" 
  data-event-category="Article Page" data-track-event="click">Papers, Zotero, 
  Reference Manager, RefWorks <SPAN>(.RIS)</SPAN></A></LI>
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=endnote&amp;flavour=full" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download EndNote citations and references" 
  data-event-category="Article Page" data-track-event="click">EndNote 
  <SPAN>(.ENW)</SPAN></A></LI>
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=bibtex&amp;flavour=full" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download Mendeley, JabRef citations and references" 
  data-event-category="Article Page" data-track-event="click">Mendeley, JabRef 
  <SPAN>(.BIB)</SPAN></A></LI></UL></DIV>
<DIV class="Panel_section">
<H4 class="Panel_sectionTitle">Article citation</H4>
<UL class="ListDownload">
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=refman&amp;flavour=citation" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" 
  data-event-category="Article Page" data-track-event="click">Papers, Zotero, 
  Reference Manager, RefWorks <SPAN>(.RIS)</SPAN></A></LI>
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=endnote&amp;flavour=citation" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download EndNote article citations" 
  data-event-category="Article Page" data-track-event="click">EndNote 
  <SPAN>(.ENW)</SPAN></A></LI>
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=bibtex&amp;flavour=citation" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download Mendeley, JabRef article citations" 
  data-event-category="Article Page" data-track-event="click">Mendeley, JabRef 
  <SPAN>(.BIB)</SPAN></A></LI></UL></DIV>
<DIV class="Panel_section">
<H4 class="Panel_sectionTitle">References</H4>
<UL class="ListDownload">
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=refman&amp;flavour=references" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks references" 
  data-event-category="Article Page" data-track-event="click">Papers, Zotero, 
  Reference Manager, RefWorks <SPAN>(.RIS)</SPAN></A></LI>
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=endnote&amp;flavour=references" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download EndNote references" 
  data-event-category="Article Page" data-track-event="click">EndNote 
  <SPAN>(.ENW)</SPAN></A></LI>
  <LI><A href="http://citation-needed.services.springer.com/v2/references/10.1186/s12859-016-1005-x?format=bibtex&amp;flavour=references" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Download Mendeley, JabRef references" 
  data-event-category="Article Page" data-track-event="click">Mendeley, JabRef 
  <SPAN>(.BIB)</SPAN></A></LI></UL></DIV></DIV></DIV></DIV>
<DIV class="SideBox"><NAV class="ToC" id="ArticleNavigation" aria-label="table of contents" 
xmlns:meta="http://www.springer.com/app/meta" xmlns="">
<DIV class="Collapse Collapse--button Collapse--buttonToC isOpen" id="collapseToC">
<H3 class="Collapse_toggle Collapse_toggle--ToC" data-toggle="collapse" 
data-event-label="Toggle" data-event-action="Toggle ToC" data-event-category="Table of Contents" 
data-track-event="click" data-target="collapseToC">Table of Contents<SPAN class="Icon Icon--arrow-down-primary"></SPAN></H3>
<UL class="Collapse_content" role="menu">
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Abs1" 
  data-event-label="Abstract" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" 
  data-track-event="click">Abstract</A></LI>
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Sec1" 
  data-event-label="Background" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" 
  data-track-event="click">Background</A></LI>
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Sec2" 
  data-event-label="Methods" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" 
  data-track-event="click">Methods</A></LI>
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Sec7" 
  data-event-label="Results and discussion" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" data-track-event="click">Results and 
  discussion</A></LI>
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Sec8" 
  data-event-label="Conclusions" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" 
  data-track-event="click">Conclusions</A></LI>
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Declarations" 
  data-event-label="Declarations" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" data-track-event="click">              
          Declarations                     </A></LI>
  <LI role="menuitem"><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#Bib1" 
  data-event-label="References" data-event-action="Link Clicked" 
  data-event-category="Table of Contents" 
  data-track-event="click">References</A></LI></UL></DIV></NAV>                    
     </DIV>
<DIV class="SideBox">
<H3 class="SideBox_title">Metrics</H3>
<DL class="SideBox_defList">
  <DIV class="placeholder" id="article-access-count-loader" data-placeholder="/placeholder/v1/article/access-count/10.1186/s12859-016-1005-x #accessCount-container" 
  data-replace="true"></DIV>
  <DIV class="placeholder" id="citation-count-loader" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s12859-016-1005-x #count-container" 
  data-replace="true"></DIV></DL>
<DIV class="placeholder" id="altmetric-loader" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s12859-016-1005-x #altmetric-container" 
data-replace="true"></DIV></DIV>
<DIV class="SideBox" id="sociaMediaShare">
<H3 class="SideBox_title">Share this article</H3>
<UL class="Share u-listReset" id="shareButtons">
  <LI class="Share_item"><BUTTON class="js-btnShareOnTwitter u-btnReset" id="shareOnTwitter" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Share Twitter" 
  data-event-category="Article Page" data-track-event="click"><SPAN class="u-srOnly">Share 
  on Twitter</SPAN>                     <SPAN 
  class="Icon Icon--twitter-brand"></SPAN>                 </BUTTON>             
  </LI>
  <LI class="Share_item"><BUTTON class="js-btnShareOnFacebook u-btnReset" id="shareOnFacebook" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Share Facebook" 
  data-event-category="Article Page" data-track-event="click"><SPAN class="u-srOnly">Share 
  on Facebook</SPAN>                     <SPAN 
  class="Icon Icon--facebook-brand"></SPAN>                 </BUTTON>            
   </LI>
  <LI class="Share_item"><BUTTON class="js-btnShareOnLinkedIn u-btnReset" id="shareOnLinkedIn" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Share LinkedIn" 
  data-event-category="Article Page" data-track-event="click"><SPAN class="u-srOnly">Share 
  on LinkedIn</SPAN>                     <SPAN 
  class="Icon Icon--linkedin-brand"></SPAN>                 </BUTTON>            
   </LI>
  <LI class="Share_item"><BUTTON class="js-btnShareOnWeibo u-btnReset" id="shareOnWeibo" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Share Weibo" 
  data-event-category="Article Page" data-track-event="click"><SPAN class="u-srOnly">Share 
  on Weibo</SPAN>                     <SPAN 
  class="Icon Icon--weibo-brand"></SPAN>                 </BUTTON>             
  </LI>
  <LI class="Share_item"><BUTTON class="js-btnShareOnGplus u-btnReset" id="shareOnGooglePlus" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Share Google+" 
  data-event-category="Article Page" data-track-event="click"><SPAN class="u-srOnly">Share 
  on Google Plus</SPAN>                     <SPAN 
  class="Icon Icon--gplus-brand"></SPAN>                 </BUTTON>             
  </LI>
  <LI class="Share_item"><BUTTON class="js-btnShareOnReddit u-btnReset" id="shareOnReddit" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Share Reddit" 
  data-event-category="Article Page" data-track-event="click"><SPAN class="u-srOnly">Share 
  on Reddit</SPAN>                     <SPAN 
  class="Icon Icon--reddit-brand"></SPAN>                 </BUTTON>             
  </LI></UL></DIV>
<DIV class="SideBox">
<H3 class="SideBox_title">See updates</H3><A href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1005-x#crossmark-dialog" 
rel="modal:open" data-event-label="10.1186/s12859-016-1005-x" data-event-action="Click CrossMark" 
data-event-category="Article Page" data-track-event="click" data-crossmark="10.1186/s12859-016-1005-x"><IMG 
class="CrossMark" id="crossmark-icon" alt="Click to get updates and verify authenticity" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/crossmark.541e56be.png"> 
        </A>         
<DIV class="u-isHidden" id="crossmark-dialog"><IFRAME width="550" height="550" 
id="crossmark-iframe" src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/realMedia.htm" 
frameborder="0"></IFRAME>         </DIV>
<DIV class="Modal" data-modal="wrap">
<DIV class="Modal_inner"><A class="Icon Icon--cross-default" id="close-10.1186/s12859-016-1005-x" 
rel="modal:close"></A>                 
<DIV class="Modal_content" data-modal="content"></DIV></DIV></DIV></DIV>
<DIV class="SideBox" id="articleSections">
<H3 class="SideBox_title">Section</H3>
<UL class="list-stacked list-stacked--2 list-stacked--small">
  <LI class="list-stacked__item"><A class="list-stacked__link" href="http://bmcbioinformatics.biomedcentral.com/articles/sections/networks-analysis" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Click section link" 
  data-event-category="Article Page" data-track-event="click">Networks 
  analysis</A>             </LI></UL></DIV>
<DIV class="SideBox">
<H3 class="SideBox_title">Other Actions</H3>
<UL class="list-stacked list-stacked--2 list-stacked--small">
  <LI class="list-stacked__item"><A href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs12859-016-1005-x&amp;volume=17&amp;issue=1&amp;title=Predicting+drug+target+interactions+using+meta-path-based+semantic+network+analysis&amp;author_name=Gang+Fu&amp;start_page=1&amp;end_page=10" 
  data-event-label="10.1186/s12859-016-1005-x" data-event-action="Order article reprints" 
  data-event-category="Article Page" data-track-event="click">Order reprint</A>  
                                 </LI></UL></DIV>
<DIV class="SideBox" id="sociaMediaAccounts">
<H3 class="SideBox_title">Follow</H3>
<UL class="Follow Follow--inline">
  <LI class="Follow_item"><A href="https://twitter.com/BMC_series" 
  data-event-label="Twitter" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--twitter-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">Follow us on Twitter</SPAN>                     </A>          
         </LI></UL></DIV></ASIDE>
<DIV class="adsbox Ad Ad--skyscraper" id="FulltextAd">
<DIV class="Ad_content">
<P class="Ad_label">Advertisement</P><SPAN id="pbgrd-43415139"></SPAN>         
</DIV></DIV></DIV></DIV></main>
<P class="footer-dynamic-legal" style="display: none;"><A title="View Legal Information" 
href="http://www.miibeian.gov.cn/">沪ICP备15051854号-3     </A> </P><FOOTER class="Footer">
<DIV class="Container">
<DIV class="Footer_inner">
<DIV class="Footer_nav" id="publisherFooter">
<DIV class="Footer_menu">
<UL>
  <LI><A href="http://www.biomedcentral.com/about/contacts">Contact us</A></LI>
  <LI><A href="http://www.biomedcentral.com/about/jobs">Jobs</A></LI>
  <LI><A href="http://www.biomedcentral.com/logon?url=/my/manuscripts">Manage 
  manuscripts</A></LI>
  <LI><A href="http://www.biomedcentral.com/logon?url=/my/preferences">Sign up 
  for article alerts</A></LI>
  <LI><A href="http://www.biomedcentral.com/logon?url=/my/preferences">Manage 
  article alerts</A></LI>
  <LI><A href="https://biomedcentral.typeform.com/to/ve8NOd">Leave 
  feedback</A></LI></UL></DIV>
<DIV class="Footer_menu">
<UL>
  <LI><A href="http://www.biomedcentral.com/about/press-centre">Press 
  center</A></LI>
  <LI><A href="http://blogs.biomedcentral.com/">Read more on our 
blogs</A></LI></UL></DIV>
<DIV class="Footer_menu">
<UL>
  <LI><A href="http://www.biomedcentral.com/about/policies">Policies</A></LI>
  <LI><A href="http://www.biomedcentral.com/licensing">Licensing</A></LI>
  <LI><A href="http://www.biomedcentral.com/terms-and-conditions">Terms and 
  conditions</A></LI>
  <LI><A href="http://www.biomedcentral.com/terms-and-conditions/privacy-statement">Privacy 
  statement</A></LI>
  <LI><A 
href="http://www.biomedcentral.com/accessibility">Accessibility</A></LI>
  <LI><A 
href="http://www.biomedcentral.com/about/cookies">Cookies</A></LI></UL></DIV>
<DIV class="Footer_menu">
<DIV class="Footer_follow">
<H3>Follow BioMed Central</H3>
<UL class="Follow Follow--inline">
  <LI class="Follow_item Follow_item--twitter"><A href="https://twitter.com/biomedcentral" 
  target="_blank" data-event-label="Twitter" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--twitter-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">Twitter</SPAN>                         </A>                   
    </LI>
  <LI class="Follow_item Follow_item--facebook"><A href="https://www.facebook.com/BioMedCentral" 
  target="_blank" data-event-label="Facebook" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--facebook-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">Facebook</SPAN>                         </A>                  
     </LI>
  <LI class="Follow_item Follow_item--gplus"><A href="https://plus.google.com/106226337984152901734/" 
  target="_blank" data-event-label="Google Plus" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--gplus-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">Google Plus</SPAN>                         </A>               
        </LI>
  <LI class="Follow_item Follow_item--youtube"><A href="https://www.youtube.com/user/BioMedCentral" 
  target="_blank" data-event-label="YouTube" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--youtube-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">YouTube</SPAN>                         </A>                   
    </LI>
  <LI class="Follow_item Follow_item--linkedin"><A href="https://www.linkedin.com/company/biomed-central" 
  target="_blank" data-event-label="LinkedIn" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--linkedin-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">LinkedIn</SPAN>                         </A>                  
     </LI>
  <LI class="Follow_item Follow_item--reddit"><A href="https://www.reddit.com/domain/biomedcentral.com" 
  target="_blank" data-event-label="Reddit" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--reddit-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">Reddit</SPAN>                         </A>                    
   </LI>
  <LI class="Follow_item Follow_item--weibo"><A href="http://www.weibo.com/biomedcentral" 
  target="_blank" data-event-label="Weibo" data-event-action="Click Follow Social Media" 
  data-event-category="Social Media" data-track-event="click"><SPAN class="Icon Icon--weibo-bordered-grey Follow_icon"></SPAN><SPAN 
  class="u-srOnly">Weibo</SPAN>                         </A>                     
  </LI></UL></DIV></DIV></DIV>
<DIV class="Footer_legal u-marginBtmL">
<P id="Test-FooterExists">© 2016 BioMed Central Ltd unless otherwise stated. 
Part of Springer Science+Business Media.                             </P>
<P>                                By continuing to use this website, you agree 
to our <A href="http://www.biomedcentral.com/terms-and-conditions">Terms and 
Conditions</A>, <A href="http://www.biomedcentral.com/terms-and-conditions/privacy-statement">Privacy 
                               statement</A> and <A href="http://www.biomedcentral.com/about/cookies">Cookies</A> 
policy.                             </P></DIV></DIV></DIV>
<DIV class="BrandingBar u-clearfix">
<DIV class="Logo BrandingBar_logo"><IMG alt="Springer Logo" src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/springer-logo_75x20.7763bac1.png" 
srcset="/static/images/springer-logo_75x20.a4e1b990.svg">                     
</DIV></DIV></FOOTER></DIV><IMG class="tracker" style="display: none;" alt="" 
src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/s12859-016-1005-x.png"> 
    
<DIV id="JFulltext"></DIV><!-- Pubgrade Ads -->     
<SCRIPT src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/bmc.js" type="text/javascript"></SCRIPT>
     <!-- End Ads -->         
<SCRIPT type="text/javascript">
        var webtrekkConfig = webtrekkConfig || {
                    trackId: '574176719203686',
                    trackDomain: "q3.webtrekk.net",
                    domain: config.site
                };

        var waitForWebtrekk = function () {
            if (typeof(webtrekkV3) === 'function') {
                setupWebtrekk();
            } else {
                setTimeout(waitForWebtrekk, 100);
            }
        };

        var setupWebtrekk = function () {
            var pageconfig = {
                contentId: "biomed_central.bmcbioinformatics_biomedcentral_com.articles.10_1186.s12859-016-1005-x",
                linkTrack: "link",
                heatmap: "1",
                form: "1"
            };

            var wt = new webtrekkV3(pageconfig);
            wt.product = '10.1186/s12859-016-1005-x';
            wt.contentGroup = {
                1: 'biomed_central',
                2: 'bmcbioinformatics_biomedcentral_com',
                3: 'Article',
                4: 'OriginalPaper'
            };
            wt.productCategory = {
                1: '12859 | BMC Bioinformatics'
            };
            wt.sendinfo();
        };

        waitForWebtrekk();
    </SCRIPT>
     <NOSCRIPT>        &lt;img 
src="//track.wt-eu02.net/574176719203686/wt?p=405,biomed_central.bmcbioinformatics_biomedcentral_com.articles.10_1186.s12859-016-1005-x&amp;cg1=biomed_central&amp;cg2=bmcbioinformatics_biomedcentral_com&amp;cg3=Article&amp;cg4=OriginalPaper&amp;ca=12859+%7C+BMC+Bioinformatics&amp;ba=10.1186%2Fs12859-016-1005-x" 
            height="1" width="1" alt=""/&gt;     </NOSCRIPT>     
<SCRIPT src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/webtrekk_v4.min.fa026a67.js" type="text/javascript" async="true"></SCRIPT>
 
<SCRIPT src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/oscar.eeb23671.js" async="true"></SCRIPT>
 
<SCRIPT src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/mathjaxConfig.698a06c9.js" async="true"></SCRIPT>
 
<SCRIPT type="text/javascript">
    if (config) {
        (function (d) {
            Array.prototype.forEach.call(d.getElementsByClassName('RealMediaAd'), function (adElement) {
                var iframe = d.createElement('iframe');
                iframe.frameBorder = '0';
                iframe.scrolling = 'no';
                var position = adElement.getAttribute("data-position"),
                        iframeInDoc = adElement.getElementsByClassName('Ad_content')[0].appendChild(iframe),
                        doc = iframeInDoc.contentWindow.document;

                doc.open().write('<body onload="window.location.href = '
                        + '\'/placeholder/v1/advertising/realMedia?siteWithPath=' + encodeURIComponent(config.siteWithPath)
                        + '&placement=' + position + '\';">');
                doc.close();
            });
        })(document);
    }
</SCRIPT>
 
<SCRIPT type="text/javascript">
    (function(e,c,a,g,f){function d(){var b=c.createElement("script");b.async=!0;
        b.src="//radar.cedexis.com/1/19501/radar.js";c.body.appendChild(b)}
        (function(){for(var b=[/\bMSIE (5|6)/i],a=b.length;a--;)if(b[a]
                        .test(navigator.userAgent))return!1;return!0})()
        &&("complete"!==c.readyState?(a=e[a])?a(f,d,!1):(a=e[g])&&a("on"+f,d):d())})
    (window,document,"addEventListener","attachEvent","load");
</SCRIPT>
             
<SCRIPT src="Predicting%20drug%20target%20interactions%20using%20meta-path-based%20semantic%20network%20analysis%20%20BMC%20Bioinformatics%20%20Full%20Text_files/entry-point.js" type="text/javascript" async=""></SCRIPT>
     </BODY></HTML>
